## Page 1

The genetic prehistory of southern Africa
Joseph K. Pickrell1*, Nick Patterson2, Chiara Barbieri3,14, Falko Berthold3,15, Linda Gerlach3,15,
Tom Güldemann4, Blesswell Kure5, Sununguko Wata Mpoloka6, Hirosi Nakagawa7, Christfried
Naumann4, Mark Lipson8, Po-Ru Loh8, Joseph Lachance9, Joanna Mountain10, Carlos
Bustamante11, Bonnie Berger8, Sarah Tishkoff9, Brenna Henn11, Mark Stoneking12, David
Reich1,2*, Brigitte Pakendorf3,13*
1Department of Genetics, Harvard Medical School, Boston
2Broad Institute of MIT and Harvard, Cambridge
3Max Planck Research Group on Comparative Population Linguistics, MPI for Evolutionary
Anthropology, Leipzig
4Seminar für Afrikawissenschaften, Humboldt University, Berlin and Department of Linguistics,
MPI for Evolutionary Anthropology, Leipzig
5Department of Aesthetics and Communication, Aarhus University, Aarhus
6Department of Biological Sciences, University of Botswana
7Institute of Global Studies, Tokyo University of Foreign Studies, Tokyo
8Department of Mathematics and Computer Science and Artificial Intelligence Laboratory, MIT,
Cambridge
9Departments of Biology and Genetics, University of Pennsylvania, Philadelphia
1023andMe, Mountain View
11Department of Genetics, Stanford University
12 Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
13Current affiliation: Laboratoire Dynamique du Langage, UMR5596, CNRS and Université Lyon
Lumière 2
14Current affiliation: Department of Evolutionary Genetics, MPI for Evolutionary Anthropology,
Leipzig
15Current affiliation: Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig and
Seminar für Afrikawissenschaften, Humboldt University, Berlin
*To whom correspondence should be addressed: joseph_pickrell@hms.harvard.edu (JKP)
reich@genetics.med.harvard.edu (DR), Brigitte.Pakendorf@ish-lyon.cnrs.fr (BP),
17 September, 2012
To appear in Nature Communications
2102
peS
71
]EP.oib-q[
2v2555.7021:viXra

## Page 2

Abstract
Southern and eastern African populations that speak non-Bantu languages with click
consonants are known to harbour some of the most ancient genetic lineages in humans, but
their relationships are poorly understood. Here, we report data from 23 populations analyzed at
over half a million single nucleotide polymorphisms, using a genome-wide array designed for
studying human history. The southern African Khoisan fall into two genetic groups, loosely
corresponding to the northwestern and southeastern Kalahari, which we show separated within
the last 30,000 years. We find that all individuals derive at least a few percent of their genomes
from admixture with non-Khoisan populations that began approximately 1,200 years ago. In
addition, the east African Hadza and Sandawe derive a fraction of their ancestry from admixture
with a population related to the Khoisan, supporting the hypothesis of an ancient link between
southern and eastern Africa.

## Page 3

Introduction
The prehistory of the populations of southern Africa that speak non-Bantu languages with click
consonants (here called Khoisan without implying linguistic unity) is poorly understood. A major
open question concerns the relationships among these populations, who harbour extensive
linguistic diversity (there are three Khoisan language families1–4) as well as variable modes of
subsistence (while most Khoisan groups are hunter-gatherers, some are pastoralists). A second
major question is the historical relationship of the southern African populations to two
populations in eastern Africa that are or previously were hunter-gatherers and that also speak
languages with click consonants (the Hadza and Sandawe). It has been hypothesized that the
eastern Africans descend in part from a Khoisan-related hunter-gatherer population that once
occupied a region ranging over much of southern and eastern Africa5. However, the
anthropological and archaeological evidence for this hypothesis is contested6,7. Apart from
shared use of click consonants, there is no linguistic evidence that the non-Bantu languages in
southern Africa and Hadza stem from a common ancestor8–10, although a potential ancestral link
between Sandawe and the Khoe-Kwadi family has been suggested4,11.
Genomic studies have the potential to shed new light on the history of the Khoisan. Previous
genetic studies based largely on single loci (mitochondrial DNA and the Y chromosome) have
documented that the Khoisan carry some of the most ancient lineages in humans12,13 and have
suggested deep genetic links between the Khoisan and the Sandawe and Hadza13. However,
single locus studies have limited resolution. While some genome-wide studies have included
southern Africans, they have largely focused on a single Khoisan group, making it impossible to
elucidate relationships among these populations14–16. The few studies of more than one Khoisan
group have not included enough populations to form a clear picture of the pattern of sub-
structure and population relationships within southern Africa17,18.
Here we present a high-resolution study of the genomic relationships of southern and eastern
African populations that speak languages characterized by heavy use of click consonants. Our
study capitalizes on three novel resources: (1) a unique collection of southern African DNA
samples encompassing most of the linguistic and cultural diversity of Khoisan groups; (2) a
single nucleotide polymorphism (SNP) array that is the first to include polymorphisms
discovered in Khoisan; and (3) new methods of statistical analysis, some of which we introduce
here for the first time, that allow us to make inferences about historical relationships even in the
presence of admixture.
Results
Data set
We genotyped 565,259 SNPs in 187 individuals from 22 African populations (16 Khoisan
populations and 5 neighboring populations speaking Bantu languages shown in Figure 1A, plus

## Page 4

the Hadza) using the Affymetrix Human Origins array19. This array is specifically designed for
studies of population history: it contains panels of SNPs discovered by sequencing a single
individual of known ancestry (including a Khoisan individual), providing precise control of the
SNP ascertainment scheme and making it possible to answer questions that are more difficult to
address using data from SNP arrays designed for medical genetics. We genotyped populations
speaking languages from all three Khoisan language families (Tuu, Kxʼa, and Khoe-Kwadi1–4,8)
(Supplementary Table S1, Supplementary Figure S1). We then merged the data with whole-
genome sequencing data from five Sandawe individuals and five Hadza individuals20. Finally,
we supplemented this with previously collected Affymetrix Human Origins array data on Dinka,
Mbuti, Biaka, Yoruba, and other African and non-African populations19,21.
The Khoisan genetically cluster into two major groups
We performed a qualitative exploration of southern African population relationships using
principal component analysis (PCA)22 (Supplementary Figures S2-S5). We capitalized on the
design of the Human Origins array by performing the analysis using three different panels of
SNPs, each of which reveals different aspects of population structure (Supplementary Figure
S4). The Yoruba SNPs highlight structure within non-Khoisan Africans, while the French SNPs
highlight European ancestry in the Nama (consistent with historic documentation23) and hint at
European or east African ancestry in some Khoe groups (Supplementary Figures S4, S6). The
SNPs ascertained in a Ju|ʼhoan individual (HGDP “San”) reveal structure invisible to the other
panels (Figure 1B). The PCA based on these SNPs divides Africans into three broad clusters: a
predominantly non-Khoisan cluster and two Khoisan clusters. The Khoisan clusters do not
correspond to linguistic affiliation; while one is comprised of Ju|ʼhoan_North and
Ju|ʼhoan_South, who speak closely related languages/dialects, the other includes populations
speaking languages belonging to all three language families (Supplementary Figure S1). The
Khoisan clusters instead reflect geography, corresponding roughly to the northwestern and
southeastern Kalahari (Figure 1A). On a fine scale, the PCA plot also identifies substructure
within individual populations (Supplementary Methods), as well as cases of discordance
between linguistic and genetic affiliation that suggest language shift with little accompanying
gene flow. A particularly striking example is the Damara, who cluster with non-Khoisan
populations despite speaking a Khoe language. This suggests that the Damara were a non-
Khoisan population who acquired their language from their Khoisan neighbours (the Nama24)
with little Khoisan gene flow25.
Admixture in southern Africa is primarily related to the Bantu Expansion
A number of populations occupy intermediate positions between the three major clusters (non-
Khoisan, northwestern Kalahari, and southeastern Kalahari) in Figure 1B. This suggests
historical gene flow; however, PCA does not constitute a formal test of admixture. We next
created a filtered dataset, excluding individuals who were outliers with respect to others from the
same self-identified ethno-linguistic group (Supplementary Table S2, Supplementary Figure S7).

## Page 5

Formal tests for a history of mixture (“three population tests”26) confirmed many examples of
population mixture (Supplementary Table S3). Most populations are admixed between a non-
Khoisan population and a population from either the northwestern Kalahari or the southeastern
Kalahari cluster (Supplementary Table S3). The one exception is the Naro, who are genetically
admixed between northwestern and southeastern Kalahari populations, just as they are
intermediate geographically (Figure 1A).
Several Khoisan populations that are at the extremes of Figure 1B—the Ju|'hoan_North,
Ju|ʼhoan_South, ǂHoan, Taa_North, and Taa_East—do not show evidence of admixture by
formal three-population tests; some of these also show no evidence of admixture in
STRUCTURE-like analyses (Supplementary Figure S8). This is intriguing because if these
populations were indeed unadmixed, they could be used as representatives of the ancestral
northwestern and southeastern Kalahari populations. However, the three-population tests have
limited power, and STRUCTURE-like methods may not be able to detect admixture if there is no
unadmixed relative in the dataset (Supplementary Figure S9). We therefore developed a novel
test for admixture that takes advantage of the fact that if and only if population mixture occurred,
we expect to detect linkage disequilibrium (LD)—non-random association of SNP genotypes—
that is correlated to the allele frequency differences between the two ancestral populations27,28
(Supplementary Figure S10). In all five populations, we observe LD that decays exponentially
with genetic distance. This is evidence that all Khoisan populations in our study, even the most
isolated, are admixed with non-Khoisan populations (Figure 2A; Supplementary Figures S11,
S12).
To estimate the proportion of admixture in the different Khoisan populations and to estimate
when it occurred, we performed a quantitative analysis of the LD decay. Using population
genetic theory presented in the Supplementary Methods, we show how the proportion of
admixture can be derived from the amplitude of the exponential curve, that is, the point from
which LD begins to decay. It has previously been shown that the time since admixture can be
derived from the rate of LD decay27,29,30, and we also use this information below. The amplitude
provides evidence of approximately 6% non-Khoisan ancestry in the Ju|'hoan_North (Figure 2A,
Supplementary Methods). We then inferred the admixture proportions in the other southern
Africans using a modified f ratio estimate19 that accounts for the admixture in the reference
4
population (Figure 2B, Supplementary Table 4, Supplementary Methods). The estimated
proportion of non-Khoisan ancestry in non-Bantu speakers ranges from 6% (Ju|'hoan_North) to
around 90% (Damara) (Figure 2B).
We next estimated the time of admixture based on the extent of the LD. Ideally we would like to
infer a distribution of times to learn when the gene flow began and when it reached its peak31,
but with current methods it is not possible to make robust statements about mixture events that
are older than a dozen generations (due to errors in inference of local ancestry30). Instead, we
estimate a single date for the gene flow, which can be thought of as the weighted average of the
admixture times27. We estimated this separately in each southern African population (Figure 2C;

## Page 6

Supplementary Table S4, Supplementary Figure S13). The earliest dates are around 40
generations (approximately 1,200 years) in the past, and the most recent dates are within the
past few hundred years (though many of the populations with recent dates show evidence of
additional gene flow before this; Supplementary Figure S14). These dates are consistent with
archeological evidence for the arrival of both east African pastoralists as well as agriculturalists
(probably Bantu speakers) in southern Africa 2,000-1,200 years ago 32–35. PCA shows that the
majority of admixture in the Khoisan is more closely related to the Yoruba (from west Africa,
linguistically related to Bantu speakers) than to the Dinka (from northeastern Africa)
(Supplementary Figure S5), though our data are consistent with additional east African ancestry
in some Khoe-speakers (Supplementary Methods).
NW and SE Kalahari Khoisan split within the last 30,000 years
To infer the date of population separation between the northwestern and southeastern Kalahari
Khoisan, we developed a new methodology enabled by the design of the Human Origins array.
The method is based on the rate at which Ju|'hoan-ascertained SNPs are observed to be
monomorphic in the other populations. The excess of monomorphic SNPs beyond that expected
due to genetic drift alone reflects new mutations that have arisen in the Ju|ʼhoan_North since
the two populations split, and thus provides a measure of the time since the split
(Supplementary Methods, Supplementary Figure S15). We verified that this approach can
provide accurate estimates of population split dates by simulation (Supplementary Figures S16,
S17), and estimated that the split of the northwestern and southeastern Kalahari Khoisan
occurred in the last 30,000 years (Supplementary Figures S18, S19). However, this date is likely
overestimated due to Bantu-related gene flow in these populations, and so should be treated as
an upper bound (Supplementary Figure S17).
A genetic link between southern and eastern African populations
We examined two eastern African populations that speak languages with click consonants
(Hadza and Sandawe) along with representative southern African populations using TreeMix36.
This method fits a population graph—a generalization of a phylogenetic tree that incorporates
the possibility of population mixture—to the allele frequency correlation patterns among a set of
sampled populations. TreeMix infers that the Hadza are admixed between a Khoisan population
(equally related to both the northwestern and southeastern Kalahari groups) and a population
most closely related to the Dinka, with about 23 ± 2% Khoisan-related ancestry (Supplementary
Figure S20). The Sandawe show a similar signal, though weaker; TreeMix estimates that the
Sandawe trace about 18 ± 2% of their ancestry to admixture with a population related to the
Khoisan (Supplementary Figure S21).
TreeMix fits a single model to a large number of populations, and in principle the finding of deep
connections between southern and eastern Africans could be an artifact of modeling a complex
history with a single admixture event. To explore the robustness of this finding, we used a four-

## Page 7

population test26 to determine whether the tree [Chimp, Ju|'hoan_North,[Hadza,Dinka]] is a good
fit to the genome-wide allele frequencies. This tree fails with a Z-score of -4.8 (p = 8 x 10-7),
indicating an excess of correlation in allele frequencies between the Ju|'hoan_North and Hadza.
A consistent signal is seen in the Sandawe, although it is weaker (Z-score of -2.1; p = 0.018).
Both the Hadza and the Sandawe show evidence of western Eurasian ancestry (perhaps
reflecting gene flow from previously-admixed neighboring populations16); the weaker signal of
relatedness between the Khoisan and the Sandawe may be due to a higher proportion of west
Eurasian ancestry in the Sandawe (Supplementary Figure S22). These findings are consistent
with the hypothesis that the Hadza and Sandawe harbor a proportion of their ancestry from a
population related to southern Africans. Alternatively, more gene flow from an (as yet
undiscovered) archaic human population into the ancestors of the Dinka than the Hadza (or
Sandawe) could produce this signal. It has been suggested that the Mbuti and Biaka
populations in central Africa may also be related to the Khoisan16,17; while our analyses show
that these populations do carry deep human lineages, they do not share the signal of
relatedness to the Khoisan that we are focusing on here (Supplementary Figure S23). In sum,
these results strongly suggest a genetic link between populations in southern and eastern Africa
that speak non-Bantu languages with heavy use of click consonants.
A unified model for the relationship of southern and eastern Africans
We used TreeMix to build a unified model for the ancestral relationships between the Khoisan
and eastern African populations, taking into account the confounding factor that all the
populations harbor recent admixture. To do this, we extended TreeMix to subtract out the effect
of gene flow from non-Khoisan populations (Supplementary Methods). This analysis provides
strong evidence for a shared origin for the Khoisan-related genetic material in the Hadza and
Sandawe. The Khoisan-related ancestry in the Hadza and Sandawe forms one clade, while the
southern African Khoisan form a second clade consisting of the northwestern and southeastern
Kalahari groups (Figure 3).
Discussion
Our analysis of diverse southern and eastern African populations has documented deep
structure in southern Africa that was previously unknown: a division between NW and SE
Kalahari groups that arose within the past 30,000 years. We have also detected admixture in all
Khoisan reflecting gene flow from Bantu-speaking agriculturalists and/or eastern African
pastoralists within the past 1,200 years. Finally, we demonstrate an ancient link between the
Khoisan and the Hadza and Sandawe in eastern Africa. This has implications for the geographic
origin of modern humans, for which both eastern and southern Africa have been proposed17,37,38.
Present-day populations in southern and eastern Africa are located on both sides of the deepest
split of the tree (Figure 3), and thus from the perspective of phylogeography, our results are
equally consistent with both of these locations as the origin of modern humans.

## Page 8

Methods
Data
The southern African samples included in this study were collected in various locations in
Botswana and Namibia as part of a multidisciplinary project, after ethical clearance by the
Review Board of the University of Leipzig and with prior permission of the Ministry of Youth,
Sport and Culture of Botswana and the Ministry of Health and Social Services of Namibia.
Approximately 2ml of saliva were collected in tubes containing 2ml of stabilizing buffer. Each
sample was genotyped on the Affymetrix Human Origins array19 and merged with additional
samples19–21 (Supplementary Methods).
The SNPs on the Human Origins array are organized into panels of SNPs discovered in
different individuals. Except where otherwise noted, we restrict ourselves to using the 150,425
autosomal SNPs discovered in a single Ju|ʼhoan_North (HGDP “San”) individual. The
exceptions to this are all ROLLOFF analyses (e.g., Figures 2A and 2C), where we used all
565,259 autosomal SNPs on the array. For analyses including the Hadza and Sandawe, some
SNPs were removed due to genotyping or sequencing errors (Supplementary Methods); the
corresponding number of Ju|ʼhoan-ascertained SNPs used when analyzing these populations
was 146,843.
Analysis of population structure and mixture
PCA was performed using smartpca22 v9003. We tested for admixture using three- and four-
population tests19. To estimate admixture dates, we used ROLLOFF v62527.
To estimate the admixture proportion of the Ju|ʼhoan_North, we binned SNPs according to the
genetic distance between them (with a bin size of 0.01 cM), and for each pair of SNPs we
calculated the linkage disequilibrium between them as well as the product of the allele frequency
differences between the Ju|ʼhoan_North and the Yoruba. In each bin, we regressed the amount
of LD against the product of allele frequency differences, and fit an exponential curve to the
resulting regression coefficients. The intercept of the fitted exponential curve is expected to be
f/(1-f), where f is the mixture fraction (see Supplementary Methods for details).
Estimating population divergence times
To date the split between the NW and SE Kalahari groups, we developed a new method based
on the fact that after the split of two populations, a given lineage from one of the populations
accumulates mutations (that are not observed in the other population) at a clock-like rate that is
proportional to years. Our method enables us to count these mutations, and convert this count
to absolute time (see Supplementary Methods for details).

## Page 9

Building population trees
To build population trees in the presence of admixture, we modified the TreeMix model36. We
first constructed a tree using unadmixed populations (Chimpanzee, Yoruba, Dinka, Europeans,
and East Asians) and then added admixed Khoisan populations to this tree using their
estimated admixture proportions (see Supplementary Methods for details).

## Page 10

Figure Legends
Figure 1: Population structure in southern Africa. A. Approximate locations of sampled
populations. Populations are colored according to linguistic affiliation, as displayed in the
legend and Figure S1. The speckled region is the Kalahari semi-desert. B. PCA on SNPs
ascertained in a Ju|'hoan individual. Shown are the positions of each individual along the first
and second axes of genetic variation, with symbols denoting the individualʼs population and
linguistic affiliation using the same color coding as in panel A.
Figure 2: All Khoisan populations are admixed. A. Admixture LD in the Ju|'hoan_North.
For each pair of SNPs in the Ju|'hoan_North (black) or the Yoruba (grey) we estimate the
linkage disequilibrium as well as the product of the differences in allele frequency between the
Ju|'hoan_North and the Yoruba. (We use the Yoruba as a proxy for the non-Khoisan,
presumably Bantu-speaking ancestral population because there has been very little change in
allele frequencies between Niger-Congo-speaking groups). We then binned pairs of SNPs by
the genetic distance between them. For each bin, we plot the regression coefficient (over SNP
pairs in the bin) from regressing the level of LD on the product of the allele frequency
differences. The rate at which this curve decays is informative about the date of admixture, while
the amplitude of the curve is informative about the proportion of admixture (Supplementary
Methods). In black is the curve if we assume the Ju|ʼhoan_North are admixed; in grey if we
assume the Yoruba are admixed (which serves as a negative control). The red line is the
exponential curve fitted to the black points. B. Estimates of mixture proportions. We used the
modified f ratio19 (Supplementary Methods) to estimate the fraction of non-Khoisan ancestry in
4
each southern African population. C. Estimates of mixture dates. We used the rate at which
admixture LD decays to estimate dates of admixture for all southern African populations
(Supplementary Methods). We plot the means, with ranges representing one standard error. Not
shown are the Wambo, who have no detectable curve, and hence may be unadmixed. The
estimates of the mixture proportions and dates are also presented in Supplementary Table S4.
Figure 3: Relationships among Khoisan and eastern Africans after removing non-Khoisan
admixture. We extended TreeMix to build trees after subtracting out the effect of known
admixture (Supplementary Methods) and then applied it to the Khoisan (excluding the Damara,
who are genetically close to non-Khoisan). Populations are coloured according to their linguistic
affiliation (Khoisan) or geographic location (dark grey = non-Khoisan African, light grey =
Eurasian), and the chimpanzee was used as an outgroup. The bar chart next to each population
shows the estimated ancestry proportions for each population: blue is the proportion of Khoisan
ancestry, and red is the proportion of non-Khoisan ancestry. Note that the actual source of these
two ancestries may vary among populations. The proportions are not identical to those
presented in Figure 2B because of small differences in how they are estimated. The black dots
show splits supported by more than 95% of bootstrap replicates, and the grey dots those
supported by more than 80% of bootstrap replicates.

## Page 11

Author Contributions
The southern African samples were collected by CB, FB, LG, TG, BK, SM, HN, CN, MS, and
BP. Hadza and Sandawe samples and genotypes were provided by ST, JL, BH, JM, and CB.
Analysis was performed primarily by JP and NP, with regular input and guidance from DR, MS,
and BP. BB, PR, and ML contributed to the method for estimating admixture levels from LD. The
study was designed by TG, DR, NP, MS and BP. The manuscript was written by JP, NP, MS,
DR, and BP, with input from all authors.
Acknowledgements
This study focuses on the prehistory of populations as reflected in their genetic variation. It does
not intend to evaluate the self-identification or cultural identity of any group, which consist of
much more than just genetic ancestry. We sincerely thank all the sample donors for their
participation in this study, the governments of Botswana and Namibia for supporting our
research, Berendt Nakwe and Justin Magabe for assistance with sample collection, Serena
Tucci, Vera Lede, Roland Schröder and Anne Butthof for assistance with sample preparation,
and Marike Schreiber for drawing Figure 1A and Supplementary Figure 1. We thank Graham
Coop, Jonathan Pritchard, Alan Barnard, and Gertrud Boden for comments on an earlier version
of this manuscript. S.W.M. thanks the University of Botswana for research leave. This work, as
part of the European Science Foundation EUROCORES Programme EuroBABEL, was
supported by grants from the Deutsche Forschungsgemeinschaft (to BP and TG), by a Grant-in-
Aid for Scientific Research (B), Ref. 19401019, Japan Society for the Promotion of Science (to
HN), as well as by funds from the Max Planck Society (to BP and MS). ST was funded by NSF
grant BCS-0827436 and NIH grants GM076637 and ES022577. JL was funded by NIH NRSA
postdoctoral fellowship HG006648. JP, NP and DR were funded by NIH grant GM100233 and
NSF HOMINID grant #1032255.

## Page 12

References
1. Heine, B. & Honken, H. The Kx’a Family: A New Khoisan Genealogy. Journal of Asian and
African Studies 79, 5–36 (2010).
2. Güldemann, T. Studies in Tuu (Southern Khoisan). (Institut für Afrikanistik, Universität
Leipzig: Leipzig, 2005).
3. Guldemann, T. Reconstruction through de-construction: The marking of person, gender, and
number in the Khoe family and Kwadi. Diachronica 21, 251–306 (2004).
4. Güldemann, Tom & Elderkin, Edward D. On external genealogical relationships of the Khoe
family. Khoisan languages and linguistics: proceedings of the 1st International Symposium
January 4-8, 2003, Riezlern/Kleinwalsertal (2010).
5. Tobias, P. V. Bushman hunter-gatherers: a study in human ecology. Ecological studies in
southern Africa 69–86 (1964).
6. Morris, A. G. The myth of the East African’Bushmen’. The South African Archaeological
Bulletin 85–90 (2003).
7. Schepartz, L. A. Who were the later Pleistocene eastern Africans? African archaeological
review 6, 57–72 (1988).
8. Sands, B. E. Eastern and southern African Khoisan: evaluating claims of distant linguistic
relationships. (R. Köppe: Cologne, 1998).
9. Güldemann, T. & Vossen, R. Khoisan. African languages: an introduction 99–122 (2000).
10. Güldemann, T. Greenberg’s ‘case’ for Khoisan: the morphological evidence. Problems of
linguistic-historical reconstruction in Africa. 19, 123–153 (2008).
11. Elderkin, E. D. Diachronic inferences from basic sentence and noun structure in Central
Khoisan and Sandawe. Tagungsberichte des Internationalen Symposions ‘Afrikanische
Wildbeuter’, Sankt Augustin, Januar 3-5, 1985 7, 131–156 (1986).
12. Knight, A. et al. African Y chromosome and mtDNA divergence provides insight into the
history of click languages. Curr Biol 13, 464–73 (2003).
13. Tishkoff, S. A. et al. History of click-speaking populations of Africa inferred from mtDNA
and Y chromosome genetic variation. Mol. Biol. Evol. 24, 2180–2195 (2007).
14. Li, J. Z. et al. Worldwide human relationships inferred from genome-wide patterns of
variation. Science 319, 1100–1104 (2008).
15. Rosenberg, N. A. et al. Genetic structure of human populations. Science 298, 2381–2385
(2002).
16. Tishkoff, S. A. et al. The genetic structure and history of Africans and African Americans.
Science 324, 1035–1044 (2009).
17. Henn, B. M. et al. Hunter-gatherer genomic diversity suggests a southern African origin for
modern humans. Proc. Natl. Acad. Sci. U.S.A. 108, 5154–5162 (2011).
18. Schuster, S. C. et al. Complete Khoisan and Bantu genomes from southern Africa. Nature
463, 943–947 (2010).
19. Patterson, N. J. et al. Ancient Admixture in Human History. Genetics
(2012).doi:10.1534/genetics.112.145037
20. Lachance, J. et al. Evolutionary History and Adaptation from High-Coverage Whole-
Genome Sequences of Diverse African Hunter-Gatherers. Cell (2012).
21. Meyer, M. et al. A High-Coverage Genome Sequence from an Archaic Denisovan
Individual. Science (2012).doi:10.1126/science.1224344

## Page 13

22. Patterson, N., Price, A. L. & Reich, D. Population structure and eigenanalysis. PLoS Genet 2,
e190 (2006).
23. Wallace, M. A History of Namibia: From the Beginning to 1990. (Columbia University
Press: 2011).
24. Barnard, A. Hunters and herders of southern Africa: a comparative ethnography of the
Khoisan peoples. 85, (Cambridge University Press: Cambridge, 1992).
25. Nurse, G. T., Lane, A. & Jenkins, T. Sero-genetic studies on the Dama of South West Africa.
Annals of Human Biology 3, 33–50 (1976).
26. Reich, D., Thangaraj, K., Patterson, N., Price, A. L. & Singh, L. Reconstructing Indian
population history. Nature 461, 489–94 (2009).
27. Moorjani, P. et al. The history of African gene flow into Southern Europeans, Levantines,
and Jews. PLoS Genet 7, e1001373 (2011).
28. Machado, C. A., Kliman, R. M., Markert, J. A. & Hey, J. Inferring the history of speciation
from multilocus DNA sequence data: the case of Drosophila pseudoobscura and close
relatives. Mol. Biol. Evol. 19, 472–488 (2002).
29. Gravel, S. Population genetics models of local ancestry. Genetics 191, 607–619 (2012).
30. Price, A. L. et al. Sensitive detection of chromosomal segments of distinct ancestry in
admixed populations. PLoS Genet 5, e1000519 (2009).
31. Pool, J. E. & Nielsen, R. Inference of historical changes in migration rate from the lengths of
migrant tracts. Genetics 181, 711–9 (2009).
32. Phillipson, D. W. African archaeology. (Cambridge University Press: Cambridge, 2005).
33. Kinahan, J. From the Beginning: The Archaeological Evidence. A History of Namibia. From
the Beginning to 1990 15–43 (2011).
34. Segobye, A. Early Farming Communities. Ditswa Mmung: The Archaeology of Botswana
101–114 (1998).
35. Reid, A., Sadr, K. & Hanson-James, N. Herding Traditions. Ditswa Mmung: The
Archaeology of Botswana 81–100 (1998).
36. Pickrell, J. K. & Pritchard, J. K. Inference of population splits and mixtures from genome-
wide allele frequency data. ArXiv e-prints (2012).at <http://arxiv.org/abs/1206.2332>
37. Ramachandran, S. et al. Support from the relationship of genetic and geographic distance in
human populations for a serial founder effect originating in Africa. Proceedings of the
National Academy of Sciences of the United States of America 102, 15942 (2005).
38. Gonder, M. K., Mortensen, H. M., Reed, F. A., De Sousa, A. & Tishkoff, S. A. Whole-
mtDNA genome sequence analysis of ancient African lineages. Molecular biology and
evolution 24, 757–768 (2007).

## Page 14

Figure 1

## Page 15

(cid:1)(cid:2)(cid:1)(cid:1) (cid:1)(cid:2)(cid:1)(cid:3) (cid:1)(cid:2)(cid:4)(cid:1) (cid:1)(cid:2)(cid:4)(cid:3) (cid:1)(cid:2)(cid:5)(cid:1)
Figure 2
(cid:7)(cid:1)(cid:2)(cid:1)
(cid:6)(cid:1)(cid:2)(cid:1)
(cid:5)(cid:1)(cid:2)(cid:1)
(cid:1)(cid:1)(cid:2)(cid:1)
(cid:8)(cid:9)(cid:10)(cid:11)(cid:12)(cid:13)(cid:14)(cid:15)(cid:16)(cid:17)(cid:18)(cid:19)(cid:20)(cid:21)(cid:12)(cid:13)(cid:10)(cid:22)
(cid:11)(cid:13)(cid:15)(cid:9)(cid:14)(cid:9)(cid:24)(cid:24)(cid:15)(cid:19)(cid:14)(cid:16)(cid:13)(cid:19)(cid:9)(cid:10)(cid:10)(cid:15)(cid:20)(cid:21)(cid:15)(cid:23)
(cid:35)(cid:2)(cid:16)(cid:35)(cid:36)(cid:37)(cid:9)(cid:38)(cid:11)(cid:26)(cid:20)(cid:15)(cid:16)(cid:39)(cid:8)(cid:16)(cid:9)(cid:13)(cid:16)(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13) (cid:25)(cid:2)(cid:16)(cid:35)(cid:36)(cid:37)(cid:9)(cid:38)(cid:11)(cid:26)(cid:20)(cid:15)(cid:16)(cid:41)(cid:20)(cid:19)(cid:41)(cid:19)(cid:20)(cid:11)(cid:9)(cid:19)(cid:13)(cid:10)
(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13)(cid:33)(cid:34)(cid:19)(cid:20)(cid:11)(cid:32) (cid:44)(cid:24)(cid:9)(cid:38)(cid:15)(cid:36)(cid:16)(cid:12)(cid:11)(cid:16)(cid:7)(cid:45)(cid:16)(cid:9)(cid:13)(cid:16)(cid:11)(cid:32)(cid:9)(cid:10)(cid:16)(cid:12)(cid:13)(cid:12)(cid:46)(cid:43)(cid:10)(cid:9)(cid:10)(cid:47)
(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13)(cid:33)(cid:48)(cid:19)(cid:26)(cid:11)(cid:32)
(cid:34)(cid:12)(cid:20)(cid:19)
(cid:49)(cid:12)(cid:12)(cid:33)(cid:34)(cid:19)(cid:20)(cid:11)(cid:32)
(cid:49)(cid:12)(cid:12)(cid:33)(cid:50)(cid:15)(cid:10)(cid:11)
(cid:49)(cid:12)(cid:12)(cid:33)(cid:51)(cid:12)(cid:10)(cid:11)
(cid:52)(cid:53)(cid:19)(cid:12)(cid:13)
(cid:25)(cid:12)(cid:13)(cid:11)(cid:26)(cid:16)(cid:27)(cid:28)(cid:16)(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13)(cid:33)(cid:34)(cid:19)(cid:20)(cid:11)(cid:32) (cid:54)(cid:55)(cid:26)(cid:26)(cid:13)
(cid:56)(cid:30)(cid:30)(cid:12)(cid:13)(cid:12)
(cid:53)(cid:12)(cid:9)(cid:30)(cid:30)(cid:19)(cid:37)
(cid:49)(cid:10)(cid:32)(cid:57)(cid:12)
(cid:42)(cid:21)(cid:12)(cid:46)(cid:12)(cid:21)(cid:12)(cid:36)(cid:9)
(cid:48)(cid:32)(cid:26)(cid:12)
(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13)(cid:33)(cid:34)(cid:19)(cid:20)(cid:11)(cid:32)(cid:16)(cid:27)(cid:28)(cid:16)(cid:25)(cid:12)(cid:13)(cid:11)(cid:26) (cid:42)(cid:32)(cid:57)(cid:15)
(cid:49)(cid:10)(cid:57)(cid:12)(cid:13)(cid:12)
(cid:18)(cid:58)(cid:26)(cid:59)(cid:26)(cid:10)(cid:32)(cid:26)
(cid:8)(cid:12)(cid:37)(cid:12)(cid:20)(cid:12)
(cid:53)(cid:9)(cid:37)(cid:58)(cid:12)
(cid:50)(cid:12)(cid:37)(cid:58)(cid:19)
(cid:1)(cid:2)(cid:1) (cid:1)(cid:2)(cid:5) (cid:1)(cid:2)(cid:6) (cid:1)(cid:2)(cid:7) (cid:1)(cid:2)(cid:60) (cid:4)(cid:2)(cid:1)
(cid:40)(cid:20)(cid:19)(cid:41)(cid:19)(cid:20)(cid:11)(cid:9)(cid:19)(cid:13)(cid:16)(cid:13)(cid:19)(cid:13)(cid:27)(cid:42)(cid:32)(cid:19)(cid:9)(cid:10)(cid:12)(cid:13)(cid:16)(cid:12)(cid:13)(cid:14)(cid:15)(cid:10)(cid:11)(cid:20)(cid:43)
(cid:62)(cid:2)(cid:16)(cid:18)(cid:9)(cid:38)(cid:11)(cid:26)(cid:20)(cid:15)(cid:16)(cid:36)(cid:12)(cid:11)(cid:9)(cid:13)(cid:21)
(cid:49)(cid:12)(cid:12)(cid:33)(cid:50)(cid:15)(cid:10)(cid:11)
(cid:49)(cid:12)(cid:12)(cid:33)(cid:51)(cid:12)(cid:10)(cid:11)
(cid:52)(cid:53)(cid:19)(cid:12)(cid:13)
(cid:56)(cid:30)(cid:30)(cid:12)(cid:13)(cid:12)
(cid:18)(cid:58)(cid:26)(cid:59)(cid:26)(cid:10)(cid:32)(cid:26)
(cid:54)(cid:55)(cid:26)(cid:26)(cid:13)
(cid:49)(cid:10)(cid:32)(cid:57)(cid:12)
(cid:48)(cid:32)(cid:26)(cid:12)
(cid:42)(cid:21)(cid:12)(cid:46)(cid:12)(cid:21)(cid:12)(cid:36)(cid:9)
(cid:49)(cid:10)(cid:57)(cid:12)(cid:13)(cid:12)
(cid:8)(cid:12)(cid:37)(cid:12)(cid:20)(cid:12)
(cid:42)(cid:32)(cid:57)(cid:15)
(cid:53)(cid:12)(cid:9)(cid:30)(cid:30)(cid:19)(cid:37)
(cid:49)(cid:12)(cid:12)(cid:33)(cid:34)(cid:19)(cid:20)(cid:11)(cid:32)
(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13)(cid:33)(cid:48)(cid:19)(cid:26)(cid:11)(cid:32)
(cid:29)(cid:26)(cid:30)(cid:31)(cid:32)(cid:19)(cid:12)(cid:13)(cid:33)(cid:34)(cid:19)(cid:20)(cid:11)(cid:32)
(cid:34)(cid:12)(cid:20)(cid:19)
(cid:53)(cid:9)(cid:37)(cid:58)(cid:12)
(cid:1) (cid:4)(cid:1) (cid:5)(cid:1) (cid:61)(cid:1) (cid:6)(cid:1) (cid:3)(cid:1)
(cid:18)(cid:9)(cid:38)(cid:11)(cid:26)(cid:20)(cid:15)(cid:16)(cid:11)(cid:9)(cid:37)(cid:15)(cid:16)(cid:17)(cid:21)(cid:15)(cid:13)(cid:15)(cid:20)(cid:12)(cid:11)(cid:9)(cid:19)(cid:13)(cid:10)(cid:16)(cid:9)(cid:13)(cid:16)(cid:41)(cid:12)(cid:10)(cid:11)(cid:22)

## Page 16

(cid:18)(cid:2)(cid:10)(cid:19)(cid:20)(cid:21)
(cid:29)(cid:8)(cid:30)(cid:28)(cid:25)(cid:10)
(cid:22)(cid:8)(cid:19)
(cid:27)(cid:2)(cid:24)(cid:28)(cid:10)(cid:19)
(cid:51)(cid:21)(cid:3)(cid:9)(cid:7)
(cid:1)(cid:3)(cid:19)(cid:31)(cid:8)
(cid:23)(cid:24)(cid:2)(cid:25)(cid:26)(cid:8)
(cid:22)(cid:8)(cid:45)(cid:46)(cid:8)
(cid:47)(cid:8)(cid:19)(cid:45)(cid:8)(cid:33)(cid:10)
(cid:44)(cid:22)(cid:24)(cid:8)(cid:19)
(cid:36)(cid:8)(cid:8)(cid:37)(cid:48)(cid:8)(cid:30)(cid:5)
(cid:47)(cid:21)(cid:25)(cid:8)
(cid:43)(cid:40)(cid:40)(cid:8)(cid:19)(cid:8)
(cid:36)(cid:30)(cid:21)(cid:33)(cid:8)
(cid:36)(cid:8)(cid:8)(cid:37)(cid:42)(cid:24)(cid:2)(cid:5)(cid:21)
(cid:36)(cid:8)(cid:8)(cid:37)(cid:38)(cid:10)(cid:30)(cid:5)
(cid:19)(cid:24)(cid:19)(cid:50)(cid:32)(cid:21)(cid:24)(cid:3)(cid:30)(cid:8)(cid:19)(cid:6)(cid:8)(cid:19)(cid:20)(cid:10)(cid:30)(cid:5)(cid:2)(cid:49) (cid:42)(cid:8)(cid:2)(cid:24)
(cid:32)(cid:21)(cid:24)(cid:3)(cid:30)(cid:8)(cid:19)(cid:6)(cid:8)(cid:19)(cid:20)(cid:10)(cid:30)(cid:5)(cid:2)(cid:49) (cid:32)(cid:21)(cid:33)(cid:10)
(cid:39)(cid:25)(cid:40)(cid:41)(cid:21)(cid:24)(cid:8)(cid:19)(cid:37)(cid:42)(cid:24)(cid:2)(cid:5)(cid:21)
(cid:39)(cid:25)(cid:40)(cid:41)(cid:21)(cid:24)(cid:8)(cid:19)(cid:37)(cid:47)(cid:24)(cid:25)(cid:5)(cid:21)
(cid:34)(cid:35)(cid:25)(cid:25)(cid:19)
(cid:22)(cid:8)(cid:3)(cid:40)(cid:40)(cid:24)(cid:9)
(cid:11)(cid:12)(cid:11)(cid:13) (cid:11)(cid:12)(cid:11)(cid:14) (cid:11)(cid:12)(cid:11)(cid:15) (cid:11)(cid:12)(cid:16)(cid:11) (cid:11)(cid:12)(cid:16)(cid:17)
(cid:1)(cid:2)(cid:3)(cid:4)(cid:5)(cid:6)(cid:7)(cid:8)(cid:2)(cid:8)(cid:9)(cid:10)(cid:5)(cid:10)(cid:2)
Figure 3

## Page 17

Supplementary Information for: The genetic prehistory of southern
Africa
Joseph K. Pickrell1,(cid:63), Nick Patterson2, Chiara Barbieri3,14, Falko Berthold3,15,
Linda Gerlach3,15, Tom Gu¨ldemann4, Blesswell Kure5, Sununguko Wata Mpoloka6,
Hirosi Nakagawa7, Christfried Naumann4, Mark Lipson8, Po-Ru Loh8, Joseph Lachance9,
Joanna L. Mountain10, Carlos D. Bustamante11, Bonnie Berger8, Sarah Tishkoff9,
Brenna M. Henn11, Mark Stoneking12, David Reich1,2,(cid:63), Brigitte Pakendorf3,13,(cid:63)
1 Department of Genetics, Harvard Medical School, Boston
2 Broad Institute of MIT and Harvard, Cambridge
3 Max Planck Research Group on Comparative Population Linguistics,
MPI for Evolutionary Anthropology, Leipzig
4 Seminar fu¨r Afrikawissenschaften, Humboldt University, Berlin and
Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig
5 Department of Aesthetics and Communication, Aarhus University, Aarhus
6 Department of Biological Sciences, University of Botswana
7 Institute of Global Studies, Tokyo University of Foreign Studies, Tokyo
8 Department of Mathematics and Computer Science and Artificial Intelligence Laboratory, MIT, Cambridge
9 Departments of Biology and Genetics, University of Pennsylvania, Philadelphia
10 23andMe, Inc., Mountain View
11 Department of Genetics, Stanford University, Palo Alto
12 Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
13 Current affiliation: Laboratoire Dynamique du Langage, UMR5596, CNRS and Universit´e Lyon Lumi`ere 2
14 Current affiliation: Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
15 Current affiliation: Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig
and Seminar fu¨r Afrikawissenschaften, Humboldt University, Berlin
(cid:63) To whom correspondence should be addressed: Brigitte.Pakendorf@ish-lyon.cnrs.fr (BP)
reich@genetics.med.harvard.edu (DR), joseph pickrell@hms.harvard.edu (JP)
November 27, 2024

## Page 18

Contents
1 Supplementary Figures 2
2 Supplementary Tables 25
3 Supplementary Methods 29
3.1 Data . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 29
3.1.1 Sampling . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 29
3.1.2 Genotyping . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 29
3.1.3 Merging data from Lachance et al. [20] . . . . . . . . . . . . . . . . . . . . . . . . . . 30
3.1.4 Filtering “outlier” individuals . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 30
3.2 Clustering analyses . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 30
3.2.1 PCA . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 31
3.2.2 ADMIXTURE analyses . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 31
3.2.3 European ancestry in the Nama. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 32
3.3 Three- and four-population tests . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 32
3.4 Using the decay of linkage disequilibrium to test for historical admixture . . . . . . . . . . . . 33
3.4.1 Motivation . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 33
3.4.2 Methods . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 33
3.4.3 Simulations . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 34
3.4.4 Application to the Khoisan . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 35
3.4.5 The f ratio test in the presence of admixed ancestral populations . . . . . . . . . . . 35
4
3.5 Estimating mixture dates with ROLLOFF . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 36
3.6 Estimating population split times . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 36
3.6.1 Motivation . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 36
3.6.2 Methods . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 37
3.6.3 Estimation of τ . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 38
3.6.4 Calibration . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 39
3.6.5 Simulations . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 40
3.6.6 Application to the Khoisan . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 40
3.7 TreeMix analyses . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 41
3.7.1 Analysis of the Hadza . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 41
3.7.2 Analysis of the Sandawe . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 42
3.7.3 West Eurasian ancestry in the Sandawe and Hadza. . . . . . . . . . . . . . . . . . . . 42
3.7.4 Modification of TreeMix to include known admixture . . . . . . . . . . . . . . . . . . . 43
3.7.5 Analysis of the Mbuti and Biaka . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 43
1

## Page 19

1 Supplementary Figures
Supplementary Figure S1: Relationships between African languages spoken by popu-
lations in this study. In bold are populations included in this study; the Hadza and Sandawe
are not shown because they are linguistic isolates.
2

## Page 20

5
0
.
0
2 0
C 0
.
P 0
5
0
.
0
-
-0.05 0.00 0.05
PC1
Hadza Nama
Biaka G||ana
Mbuti G|ui
Himba Ju|'hoan_North
Wambo Ju|'hoan_South
Dinka !Xuun
Yoruba ǂHoan
BantuSouthAfrica Taa_West
BantuKenya Taa_North
Mbukushu Taa_East
Mandenka French
Tswana Han
Kgalagadi Oroqen
Naro Basque
Damara
Hai||om
Shua
Tshwa
Supplementary Figure S2: PCA including non-African populations. We performed prin-
cipal component analysis on the genotype matrix of individuals using smartpca [22] using the SNPs
ascertained in a Ju|’hoan North individual. Plotted are the positions of each individual along prin-
cipal component axes one and two. The colors and symbols for each population are depicted in the
legend. 3

## Page 21

0
1
.
0
5
0
.
0
0
0
.
0
2
C
P
0
1
.
0
-
0
2
.
0
-
-0.05 0.00 0.05 0.10
PC1
Himba Khwe
Wambo Shua
Dinka Tshwa
Yoruba Nama
BantuSouthAfrica G||ana
BantuKenya G|ui
Mbukushu Ju|'hoan_North
Mandenka Ju|'hoan_South
Tswana !Xuun
Kgalagadi ǂHoan
Naro Taa_West
Damara Taa_North
Hai||om Taa_East
Supplementary Figure S3: PCA of African populations using all the SNPs on the chip.
Each individual is represented by a point, and the color and style of the point is displayed in the
caption.
4

## Page 22

A. PCA (Ju|'hoan ascertainment)
5
1
Himba Naro 0.
Wambo Damara
Dinka Hai||om
Yoruba Khwe
BantuSouthAfrica Shua 5
0
BantuKenya Tshwa 0.
Mbukushu Nama
Mandenka G||ana C 2
Tswana G|ui P
Kgalagadi Ju|'hoan_North 5
0
Ju|'hoan_South 0.
!Xuun -
ǂHoan
Taa_West
Taa_North 5
Taa_East 1
0.
-
-0.05 0.00 0.05 0.10
PC1
B. PCA (Yoruba ascertainment) C. PCA (French ascertainment)
4
0.
1
0.
3
0.
0
0. 2
2 2 0.
C C
P P
1
-
0.
0.
1
2 0
0. 0.
-
-0.05 0.00 0.05 0.10 -0.05 0.00 0.05 0.10
PC1 PC1
Supplementary Figure S4: PCA on SNPs from different ascertainment panels. In each
panel, each point represents an individual. The color and style of each point corresponds to the
population of the individual as displayed in the legend. A. Ju|’hoan North ascertainment.
This is same data presented in Figure 1B in the main text, but is included for comparison. B.
Yoruba ascertainment. C. French ascertainment
5

## Page 23

1
0.
0
0.
1
C
2 0.
P -
2
0.
-
3
0.
-
-0.1 0.0 0.1 0.2
PC1
Himba Khwe
Wambo Shua
Dinka Tshwa
Yoruba G||ana
BantuSouthAfrica Ju|'hoan_North
BantuKenya Ju|'hoan_South
Mbukushu !Xuun
Mandenka ǂHoan
Tswana Taa_West
Kgalagadi Taa_North
Naro Taa_East
Damara
Hai||om
Supplementary Figure S5: PCA projection using Ju|’hoan North, Yoruba, and Dinka.
We identified principal components using only the Ju|’hoan North, Yoruba, and Dinka, then pro-
jected the other samples (excluding outliers) onto these axes. All Khoisan populations fall on a
cline between the Ju|’hoan North and the neighboring Bantu-speaking populations. This is consis-
tent with the variation in non-Khoisan admixture in these populations being due to variation in
admixture with neighboring agriculturist populations.
6

## Page 24

A. Tests of treeness B. Admixture LD
Nama Nama
Shua Shua
Hai||om
Khwe
Taa_West 0 8
Mbukushu 0.
G||ana n
Tswana o
Damara ati
!Xuun el
Himba r
Tshwa o r 4
ǂ
T
H
a
o
a
a
_
n
North
C 0. 0
Kgalagadi
Naro
Wambo
Ju|'hoan_North
Taa_East 0
Ju|'hoan_South 0
0.
-0.0005 0.0000 0.0005 0.0010 0.0015 0.00 0.05 0.10 0.15 0.20 0.25 0.30
f (Yoruba,X;Han,French) Distance (Morgans)
4
Supplementary Figure S6: The Nama have recent non-African ancestry. A.
Four-population tests. We performed four-population tests on the tree topology
[[Yoruba,X],[Han,French]], where X represents any southern African population. Plotted is the
value of the f statistic when each southern African population is used. Error bars show a single
4
standard error, and points in red have a Z-score greater than 3. B. Admixture LD. We ran
ROLLOFF on the Nama and the Shua using the Ju|’hoan North and the French as the mixing
populations. There is a clear decay in the Nama (the shift away from the x-axis is indicative of
variable ancestry across individuals, which is visually apparent in Supplementary Figure S2) and a
less obvious decay in the Shua. The red lines show the fitted exponential curves.
7

## Page 25

lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll l ll l l lll lll l l l l llll lll lll l l lll l l
lllllllllll l
l
l l
l l
ll l lll l l
l
ll l l l
ll
l l lll l
ll
ll
ll
l l l ll
l
l
l
l l l llllll ll l l
l l
ll
l l
l
l
l
l
l l
ll
lll
llllllllll
l l ll
l
l llll l l l ll lll l l l
−0.05 0.00 0.05 0.10
01.0
00.0
51.0−
Ju|'hoan_South
PC1
2CP ll lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll l ll l l lll lll l l l l llll lll lll l l lll l l
lllllllllll l
l
l l
l l
ll l lll l l
l
ll l l l
ll
l l lll l
ll
ll
ll
l l l ll
l
l
l
l l l llllll ll l l
l l
ll
l l
l
l
l
l
l l
llllll
llll
ll
lll
llllllllll
l l ll
l
l llll l l l ll lll l l l
−0.05 0.00 0.05 0.10
01.0
00.0
51.0−
Taa_North
PC1
2CP
ll ll lll
lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll l ll l l lll lll l l l l llll lll lll l l lll l l
lllllllllll l
l
l l
l l
ll l lll l l
l
ll l l l
ll
l l lll l
ll
ll
ll
l l l ll
l
l
l
l l l llllll ll l l
l l
ll
l l
l
l
l
l
l l
ll
lll
llllllllll
l l ll
l
l llll l l l ll lll l l l
−0.05 0.00 0.05 0.10
01.0
00.0
51.0−
Shua
PC1
2CP
ll
l lllll lll
lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll l ll l l lll lll l l l l llll lll lll l l lll l l
lllllllllll l
l
l l
l l
ll l lll l l
l
ll l l l
ll
l l lll l
ll
ll
ll
l l l ll
l
l
l
l l l llllll ll l l
l l
ll
l l
l
l
l
l
l l
ll lll llllllllll
l l ll
l
l llll l l l ll lll l l l
−0.05 0.00 0.05 0.10
01.0
00.0
51.0−
!Xuun
PC1
2CP l ll lll lll l ll
ll
lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll l ll l l lll lll l l l l llll lll lll l l lll l l
lllllllllll l
l
l l
l l
ll l lll l l
l
ll l l l
ll
l l lll
l
l
l ll
ll
ll
l l l ll
l l
l
ll l
l l l llllll ll l l
l l
ll
l l l
l
l
l
l lllllllll
l l ll
l
l lll
l
l l l l ll
l
lll l l l
−0.05 0.00 0.05 0.10
01.0
00.0
51.0−
Kgalagadi
PC1
2CP llll
ll
lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll l ll l l lll lll l l l l llll lll lll l l lll l l
lllllllllll l
l
l l
l l
ll l lll l l
l
ll l l l
ll
l l lll l
ll
ll
ll
l l l ll
l
l
l
l l l llllll ll l l
l l
ll
l l
l
l
l
l
l l
ll lll llllllllll
l l ll
l
l llll l l l ll lll l l l
−0.05 0.00 0.05 0.10
01.0
00.0
51.0−
Naro
PC1
2CP
l ll
ll l l ll ll
Supplementary Figure S7: Individuals excluded from populations. Shown are the PCA
plot in Figure 1 in the main text, with different populations highlighted. In red are the individuals
we excluded, and in green those that were kept. See Supplementary Table S1 for total sample sizes
in each population.
8

## Page 26

ancestry
0.0 0.4 0.8
BantuKenya
BantuSouthAfrica
Dinka
Himba
Kgalagadi
Mandenka
Mbukushu
Wambo
Tswana
Yoruba
YRI
Hadza
Hadza_TISH
Sandawe
Biaka
Mbuti
Pygmy_TISH
Damara
G||ana
G|ui
Hai||om
Khwe
Tshwa
Nama
Naro
Shua
!Xuun
ǂHoan
Ju|'hoan_North
Ju|'hoan_South
Taa_East
Taa_North
Taa_West
Basque
French
Han
Oroqen
K
=
2
ancestry
0.0 0.4 0.8
K
=
4
ancestry
0.0 0.4 0.8
K
=
5
ancestry
0.0 0.4 0.8
K
=
6
ancestry
0.0 0.4 0.8
K
=
8
SupplementaryFigureS8: Clusteringanalysesofthemergedsequencedandgenotyped
samples. We ran ADMIXTURE on all African individuals using different settings of K; shown
are the resulting clusters. The populations merge9d from Lachance et al. [20] are YRI (compare to
Yoruba), Hadza TISH (compare to Hadza), Pygmy TISH, and Sandawe.

## Page 27

A. Simulated demography
10%
pop1 pop2
yrtsecnA
0.1
8.0
6.0
4.0
2.0
0.0
Simulation 1
B. ADMIXTURE (K=2)
pop1 pop2
0.00 0.05 0.10 0.15 0.20
01.0
60.0
20.0
20.0−
Distance (Morgans)
tneiciffeoc
noissergeR
C. Decay of LD
pop1−>pop2
pop2−>pop1
yrtsecnA
0.1
8.0
6.0
4.0
2.0
0.0
Simulation 2
D. ADMIXTURE (K=2)
pop1 pop2
0.00 0.05 0.10 0.15 0.20
21.0
80.0
40.0
00.0
Distance (Morgans)
tneiciffeoc
noissergeR
E. Decay of LD
pop1−>pop2
pop2−>pop1
Supplementary Figure S9: LD information identifies previously undetectable admix-
ture events. We performed simulations of two populations, one of which admixed with the other
40 generations in the past (see Section 3.4 for details). Shown are results from two simulations.
A. The simulated demography. B,D. Results from running ADMIXTURE on the simulated data.
C,E. Results from the measure of LD decay described in Section 3.4.
10

## Page 28

0.00 0.05 0.10 0.15 0.20
03.0
52.0
02.0
51.0
01.0
50.0
00.0
Distance (Morgans)
tneiciffeoc
noissergeR
0% (sim)
0% (theory)
10% (sim)
10% (theory)
20% (sim)
20% (theory)
observed
Supplementary Figure S10: Estimating mixture proportions from LD. We simulated
genetic data under different demographies including admixture (Supplementary Information), and
then estimated admixture proportions using the method described in the text. The colored and
grey points represent the decay curve obtained in simulations (each curve is the average of five
simulations of 100 Mb), and the lines are the theoretical curves. In black is the data from the
Ju|’hoan North and Yoruba, treating the Ju|’hoan North as admixed.
11

## Page 29

0 A. Ju|'hoan_North 0 B. Ju|'hoan_South 0 C. ǂHoan
e
nt 0. 2 B
Ju
a
|
n
'h
tu
o a
-
n
>
_
J
N
u
o
|'
r
h
t
o
h
a
-
n
>
_
B
N
a
o
n
rt
t
h
u e
nt 0. 2 B
Ju
a
|
n
'h
tu
o a
-
n
>
_
J
S
u
o
|'
u
h
t
o
h
a
-
n
>
_ S
B
o
a
u
n
t
t
h
u e
nt 0. 2 B
ǂH
an
o
t
a
u
n
- >
->
ǂ
B
H
a
o
n
a
t
n
u
ci ci ci
effi effi effi
o 0 o 0 o 0
c 1 c 1 c 1
n 0. n 0. n 0.
o o o
si si si
s s s
e e e
R e
gr
0. 0
0
R e
gr
0. 0
0
R e
gr
0. 0
0
0.00 0.05 0.10 0.15 0.20 0.00 0.05 0.10 0.15 0.20 0.00 0.05 0.10 0.15 0.20
Distance (Morgans) Distance (Morgans) Distance (Morgans)
0 D. Taa_North 0 E. Taa_East
e
nt 0. 2 B
Ta
a
a
n
_
tu
N
-
o
>
r t
T
h
a
-
a
>
_
B
N
a
o
n
rt
t
h
u e
nt 0. 2 B
Ta
a
a
n
_
tu
E
-
a
>
s t
T
-
a
>
a
B
_E
an
a
t
s
u
t
ci ci
effi effi
o 0 o 0
c 1 c 1
n 0. n 0.
o o
si si
s s
e e
R e
gr
0. 0
0
R e
gr
0. 0
0
0.00 0.05 0.10 0.15 0.20 0.00 0.05 0.10 0.15 0.20
Distance (Morgans) Distance (Morgans)
Supplementary Figure S11: Admixture LD in populations that pass the three-
population test. We measured the decay of admixture LD on the five Khoisan populations
that show no evidence of admixture in three-population tests. The method is described in Section
3.4. Each panel shows an individual population; panel A. is a version of Figure 2A from the main
text with the y-axis modified to be the same as the other panels. In all cases, the non-Khoisan
population used in the analysis is the Yoruba. In red is the fitted exponential curve.
12

### Table 1 (Page 29)

| Bantu -> Ju|'hoan_South
Ju|'hoan_South -> Bantu |
| --- |
|  |

## Page 30

0 5 10 15 20
60.0
40.0
20.0
00.0
Distance (cM)
noitalerroC
Khwe −> Ju|'hoan_North<− Dinka
0 5 10 15 20
60.0
40.0
20.0
00.0
Distance (cM)
noitalerroC
Yoruba −> Ju|'hoan_North<− French
0 5 10 15 20
60.0
40.0
20.0
00.0
Distance (cM)
noitalerroC
Yoruba −> Ju|'hoan_North<− Taa_East
0 5 10 15 20
60.0
40.0
20.0
00.0
Distance (cM)
noitalerroC
French −> Ju|'hoan_North<− Han
Supplementary Figure S12: ROLLOFF analysis of the Ju|’hoan North. We explored
the correlation between the decay of LD in the Ju|’hoan North and the divergence between other
pairs of populations using ROLLOFF. At each pair of SNPs, we estimate the amount of LD in the
Ju|’hoan North (as measured by a correlation in genotypes [27] ) and the product of the differences
in allele frequency between two reference populations. The reference populations in each panel are
listed to either side of the Ju|’hoan North. We then calculate the correlation between these two
values, binning pairs of SNPs by the genetic distance between them. Each point is the value of this
correlation (the y-axis) plotted against the genetic distance bin (the x-axis). A detectable curve
suggests that the target population (in this case the Ju|’hoan) is admixed. Note a curve can be
present even if the reference populations are quite distant from the true mixing populations. In
this case, a curve is seen except when using two non-African populations as references. In red is
the fitted exponential curve.
13

## Page 31

Taa_East ǂHoan Mbukushu
Taa_West G||ana !Xuun
8 8 8
0 0 0
n 0. n 0. n 0.
o o o
ati ati ati
el el el
orr
0
4 orr
0
4 orr
0
4
C 0. C 0. C 0.
0 0 0
0 0 0
0. 0. 0.
0 5 10 15 20 25 30 0 5 10 15 20 25 30 0 5 10 15 20 25 30
Distance (cM) Distance (cM) Distance (cM)
Tshwa Kgalagadi Damara
Shua Tswana Khwe
8 8 8
0 0 0
n 0. n 0. n 0.
o o o
ati ati ati
el el el
orr
0
4 orr
0
4 orr
0
4
C 0. C 0. C 0.
0 0 0
0 0 0
0. 0. 0.
0 5 10 15 20 25 30 0 5 10 15 20 25 30 0 5 10 15 20 25 30
Distance (cM) Distance (cM) Distance (cM)
Hai||om Ju|'hoan_South Himba
Taa_North Naro Wambo
8 8 8
0 0 0
n 0. n 0. n 0.
o o o
ati ati ati
el el el
orr
0
4 orr
0
4 orr
0
4
C 0. C 0. C 0.
0 0 0
0 0 0
0. 0. 0.
0 5 10 15 20 25 30 0 5 10 15 20 25 30 0 5 10 15 20 25 30
Distance (cM) Distance (cM) Distance (cM)
Supplementary Figure S13: ROLLOFF analysis of all southern African populations.
ForeachsouthernAfricanpopulation, weranROLLOFF[27]usingtheJu|’hoan NorthandYoruba
as the mixing populations. The method is as described in Supplementary Figure S12 and the
Supplementary Material. Shown are the resulting curves for each population; in red are the fitted
exponential curves.
14

## Page 32

4 4 4
0 0 0
0. Taa_East 0. ǂHoan 0. Mbukushu
Taa_West G||ana !Xuun
n n n
o 2 o 2 o 2
el
ati
0.
0
el
ati
0.
0
el
ati
0.
0
orr
0
orr
0
orr
0
c 0 c 0 c 0
al
0.
al
0.
al
0.
u u u
d d d
si si si
e e e
R R R
4 4 4
0 0 0
0. 0. 0.
- - -
0 5 10 15 20 25 30 0 5 10 15 20 25 30 0 5 10 15 20 25 30
Distance (cM) Distance (cM) Distance (cM)
4 4 4
0 0 0
0. Tshwa 0. Kgalagadi 0. Damara
Shua Tswana Khwe
n n n
o 2 o 2 o 2
el
ati
0.
0
el
ati
0.
0
el
ati
0.
0
orr
0
orr
0
orr
0
c 0 c 0 c 0
al
0.
al
0.
al
0.
u u u
d d d
si si si
e e e
R R R
4 4 4
0 0 0
0. 0. 0.
- - -
0 5 10 15 20 25 30 0 5 10 15 20 25 30 0 5 10 15 20 25 30
Distance (cM) Distance (cM) Distance (cM)
4 4 4
0 0 0
0. Hai||om 0. Ju|'hoan_South 0. Himba
Taa_North Naro Wambo
n n n
o 2 o 2 o 2
el
ati
0.
0
el
ati
0.
0
el
ati
0.
0
orr
0
orr
0
orr
0
c 0 c 0 c 0
al
0.
al
0.
al
0.
u u u
d d d
si si si
e e e
R R R
4 4 4
0 0 0
0. 0. 0.
- - -
0 5 10 15 20 25 30 0 5 10 15 20 25 30 0 5 10 15 20 25 30
Distance (cM) Distance (cM) Distance (cM)
Supplementary Figure S14: Residuals from ROLLOFF analysis of all southern African
populations. Plotted are the residuals from the fit of the exponential curves for each population
from Supplementary Figure S13. Residual correlation may indicate multiple waves of mixture in
some populations.
15

## Page 33

A. Model demography
*
* *
A
t
*
*
O X YY
l
l
l
l
l
l
l
l
l
l l l l l l l l l
l
l
l
derived allele frequency
ytisned
B. Allelic spectrum in A (ancestral population)
l
l l l l l l l l l l l l l l l l l l l l
0 1
derived allele frequency
ytisned
C. Allelic spectrum in X
0 1
Supplementary Figure S15: Scheme for dating population splits. A. Demographic
model. Plotted is the demographic model used in our method for dating population split times.
Populations are labeled in black, and the split time is denoted t. In grey is the history of the two
chromosomes used for SNP ascertainment. Stars represent mutations, and are colored according
to whether they arose before (black) or after (red) the population split. B. A hypothetical allelic
spectrum in population A. The red peak at zero corresponds to the mutations that happened on
the lineage to Y. C. The hypothetical allelic spectrum in X. Though alleles change frequency from
A to X, the size of the red component of the peak at zero stays constant.
16

### Table 1 (Page 33)

| l |
| --- |
|  |

### Table 2 (Page 33)

| l |
| --- |
|  |

## Page 34

ll
lll
500 1000 1500 2000
0002
0051
0001
005
Simulated split time (generations)
)snoitareneg(
emit
tilps
detamitsE
*ll
l
l
l
ll
*l
l
l
*
Supplementary Figure S16: Estimating split times in simulations without migration.
Shown are the simulated and inferred split times in simulations without migration. The red stars
show the true simulated values, and the black points the estimates.
17

## Page 35

ll
lll
0.0 0.1 0.2 0.3 0.4 0.5
0002
0051
0001
005
admixture proportion in X
)snoitareneg(
emit
tilps
detamitsE
A. t = 400 generations
l l
lll
l
llll llll l l l ll l lll l l
l
lll
l
* * * * * *
0.0 0.1 0.2 0.3 0.4 0.5
0002
0001
005
admixture proportion in X
)snoitareneg(
emit
tilps
detamitsE
B. t = 2000 generations
*
l
l
ll
l * *l
llll
*
lllll
*
l
lll
l
*
l
l
l
l
l
Supplementary Figure S17: Estimating split times in simulations with migration.
Shown are estimated split times between X and Y when both have experienced some level of
admixture with an outgroup. The black points show estimated split times in the presence of ad-
mixture. The red stars show the true simulated values. In all simulations, population Y has 5%
admixture from the outgroup that occurred 40 generations in the past, while population X has
variable levels of admixture (plotted on the x-axis).
18

## Page 36

0
0
5
1
)
h
t
r
o
N
_
n
a G||ana
h o 0 0 ǂHoan
u
|'
1
0
Taa_East
J Tshwa
m
o
f r Taa_North Shua
t
pli
s
e Taa_West
c
n
si Khwe
s 0 Hai||om
n 0
o 5
ti
a
r
e !Xuun
n
e
g
(
t Naro
NW Kalahari
SE Kalahari
Ju|'hoan_South
0 unclear
0.0 0.1 0.2 0.3 0.4 0.5 0.6
fraction non-Khoisan ancestry
Supplementary Figure S18: Dating the split time of the Khoisan populations. We plot
the estimated times since each Khoisan population split from the Ju|’hoan North, as a function
of their level of non-Khoisan-admixture. The populations included are all the southern African
groups in Figure 3 in the main text. The errors bars are one standard error (not including the
error in the estimate of τ). Khoisan populations are colored according to whether they have strong
evidence (from Figure 3 in the main text) as coming from the northwestern Kalahari cluster or the
southeastern Kalahari cluster. Populations that have no clear grouping are colored in green. All
split times are likely overestimated due to non-Khoisan admixture (see Supplementary Figure S17)
19

### Table 1 (Page 36)

| G||ana
ǂHoan
Taa_East
Tshwa
Taa_North
Shua
Taa_West
Khwe
Hai||om
!Xuun
Naro
NW Kalahari
SE Kalahari
Ju|'hoan_South
unclear |  |
| --- | --- |
|  | NW Kalahari
SE Kalahari
unclear |

## Page 37

l
l
l
l
l
l
l
l l l l
0 2 4 6 8 10
02.0
01.0
00.0
Derived allele count
noitcarF
A. Fitted and empirical allelic spectra in the Taa_North
l emprical spectrum l
l l fitted (t = 0)
l l
l l
l l
l l
l l
l l
l l l l l l l l
0 2 4 6 8 10
02.0
01.0
00.0
Derived allele count
noitcarF
l l emprical spectrum
l fitted (t estimated)
l
l
l
l
l
l
l l l l
l
l l l l l l l l l l l l l l l l l l l l
0 5 10 15 20
51.0
01.0
50.0
Derived allele count
noitcarF
B. Fitted and empirical allelic spectra in a simulation of t = 2000 generations
l
l emprical spectrum
l fitted (t = 0) l
l
l
l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l l
0 5 10 15 20
51.0
01.0
50.0
Derived allele count
noitcarF
l
l emprical spectrum
l fitted (t estimated)
l l l l l l l l l l l l l l l l l l l l
Supplementary Figure S19: Dating the split time of Taa North. A.We plotthe empirical
allele frequency spectrum in the Taa North at SNPs ascertained in a single Ju|’hoan North individ-
ual (in black). For comparison we plot the fitted allelic spectra if we assume the split time between
Taa North and the Ju|’hoan North is zero (in grey in the left panel) or if we allow the model to
estimate the split time (in red in the right panel). Note that the empirical spectrum is non-linear,
implying that the ancestral population was not of constant size. B. We plot the analogous spectra
for a single simulation of a split time of 2,000 generations with no migration.
20

### Table 1 (Page 37)

| l l emprical spectrum
l l fitted (t = 0)
ll
ll
l
l
l
l
ll
ll
ll
ll
ll ll | l emprical spectrum
l fitted (t = 0) |
| --- | --- |

### Table 2 (Page 37)

| ll l emprical spectrum
l fitted (t estimated)
l
l
ll
ll
ll
ll
ll
ll
ll
ll ll | l emprical spectrum
l fitted (t estimated) |
| --- | --- |

### Table 3 (Page 37)

| l
l emprical spectrum
l fitted (t = 0)
l
l
l
l l l l ll ll
ll
ll
ll ll ll
ll ll ll ll ll
ll ll ll ll ll | l emprical spectrum
l fitted (t = 0) |
| --- | --- |

### Table 4 (Page 37)

| ll
l emprical spectrum
l fitted (t estimated)
ll ll ll ll ll
ll ll
ll ll ll ll ll ll ll ll ll ll ll ll ll | l emprical spectrum
l fitted (t estimated) |
| --- | --- |

## Page 38

A. Maximum likelihood tree B. Residual fit from tree in A.
Chimp
Chimp
Hadza 18.1 SE
Taa_East
Yoruba
Taa_North
Dinka Ju|'hoan_North
Han Ju|'hoan_South
Oroqen Yoruba
-18.1 SE
French Dinka
Basque Hadza
Taa_East French
Oroqen
Taa_North
Basque
Ju|'hoan_South
10 s.e. Han
Ju|'hoan_North
0.00 0.0 D 4 rift par 0 a . m 08 eter 0.12 C hi m
p
T a a _ E a
st
T a a _ N ort
h
J
u|' h o a n _ N ort
h
J
u|' h o a n _ S o ut
h
Y or u b
a
Di n k
a
H a d z
a
Fr e n c
h
Or o q e
n
B a s q u
e
H a
n
C. Maximum likelihood graph D. Residual fit from graph in C.
Chimp
Chimp
Yoruba 5.2 SE
Taa_East
Dinka
Taa_North
Hadza Ju|'hoan_North
Han Ju|'hoan_South
Oroqen Yoruba
Migration -5.2 SE
weig 0 h . t 5 French Dinka
Basque Hadza
Taa_East French
0 Oroqen
Taa_North
Basque
Ju|'hoan_South
10 s.e. Han
Ju|'hoan_North
0.00 0.0 D 4 rift par 0 a . m 08 eter 0.12 C hi m
p
T a a _ E a
st
T a a _ N ort
h
J
u|' h o a n _ N ort
h
J
u|' h o a n _ S o ut
h
Y or u b
a
Di n k
a
H a d z
a
Fr e n c
h
Or o q e
n
B a s q u
e
H a
n
Supplementary Figure S20: TreeMix analysis of the Hadza. Shown is the maximum
likelihood tree of populations including the Hadza (A.), the residual fit from this tree (B.), the
inferred graph allowing for a single migration edge (C.), and the residual fit from this graph (D.).
See Supplementary text for discussion. Note that the choice of which edge to the Hadza is called
the “migration” edge is arbitrary [36] ; for Figure 3 in the main text we force the non-Khoisan
ancestry in the Hadza to be the “migration” edge.
21

### Table 1 (Page 38)

|  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |

### Table 2 (Page 38)

|  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |

## Page 39

A. Maximum likelihood tree B. Residual fit from tree in A.
Chimp
Chimp
Oroqen 11.7 SE
Taa_East
Han
Taa_North
French Ju|'hoan_North
Basque Ju|'hoan_South
Sandawe Yoruba
-11.7 SE
Dinka Dinka
Yoruba Sandawe
Taa_East French
Oroqen
Taa_North
Basque
Ju|'hoan_South
10 s.e. Han
Ju|'hoan_North
0.00 0.0 D 4 rift par 0 a . m 08 eter 0.12 C hi m
p
T a a _ E a
st
T a a _ N ort
h
J
u|' h o a n _ N ort
h
J
u|' h o a n _ S o ut
h
Y or u b
a
Di n k
a
S a n d a w
e
Fr e n c
h
Or o q e
n
B a s q u
e
H a
n
C. Maximum likelihood graph D. Residual fit from graph in C.
Chimp
Chimp
Oroqen 5.5 SE
Taa_East
Han
Taa_North
French Ju|'hoan_North
Basque Ju|'hoan_South
Sandawe Yoruba
Migration -5.5 SE
weig 0 h . t 5 Dinka Dinka
Yoruba Sandawe
Taa_East French
0 Oroqen
Taa_North
Basque
Ju|'hoan_South
10 s.e. Han
Ju|'hoan_North
0.00 0.0 D 4 rift par 0 a . m 08 eter 0.12 C hi m
p
T a a _ E a
st
T a a _ N ort
h
J
u|' h o a n _ N ort
h
J
u|' h o a n _ S o ut
h
Y or u b
a
Di n k
a
S a n d a w
e
Fr e n c
h
Or o q e
n
B a s q u
e
H a
n
Supplementary Figure S21: TreeMix analysis of the Sandawe. Shown is the maximum
likelihood tree of populations including the Sandawe individuals. (A.), the residual fit from this
tree (B.), the inferred graph allowing for a single migration edge (C.), and the residual fit from
this graph (D.). See Supplementary text for discussion. Note that the choice of which edge to the
Sandawe is called the “migration” edge is arbitrary [36] ; for Figure 3 in the main text we force the
non-Khoisan ancestry in the Sandawe to be the “migration” edge.
22

### Table 1 (Page 39)

|  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |

### Table 2 (Page 39)

|  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |  |

## Page 40

5 10 15 20
30.0
10.0
10.0−
Genetic distance (cM)
noitalerroc
dethgieW
Dinka−> Hadza <− French
5 10 15 20
30.0
10.0
10.0−
Genetic distance (cM)
noitalerroc
dethgieW
Dinka−> Sandawe <− French
Supplementary Figure S22: Admixture linkage disequilibrium in the Hadza and San-
dawe. WeranROLLOFF[27]ontheHadzaandSandawe, usingtheDinkaandFrenchasreference
populations. Shown are the resulting curves for the Hadza and Sandawe. There is a striking curve
of admixture LD in the Sandawe, which is weaker in the Hadza.
23

## Page 41

Chimp
Mbuti
Biaka
Han
Oroqen
French
Basque
Dinka
Yoruba
Khwe
Ju|'hoan_South
Ju|'hoan_North
!Xuun
Hai||om
Naro
Taa_East
ǂHoan
Shua
G||ana
Tshwa
Taa_North
10 s.e.
Taa_West
0.00 0.02 0.04 0.06 0.08 0.10 0.12
Drift parameter
Supplementary Figure S23: TreeMix analysis including the Mbuti and Biaka. Weuseda
modified TreeMix approach to build a tree of populations after subtracting out Bantu or Dinka-like
ancestry. Shown is the resulting tree; see the Supplementary text for details and discussion.
24

## Page 42

2 Supplementary Tables
Population Language family Linguistic subgroup # of samples
Taa East Tuu Taa-Lower Nossob 6
Taa North Tuu Taa-Lower Nossob 6
Taa West Tuu Taa-Lower Nossob 8
!Xuun Kx’a Northwest Ju 13
Ju|’hoan North Kx’a Southeast Ju 16
Ju|’hoan South Kx’a Southeast Ju 9
}Hoan Kx’a }Hoan 7
Shua Khoe-Kwadi East Kalahari Khoe 10
Tshwa Khoe-Kwadi East Kalahari Khoe 10
Khwe Khoe-Kwadi West Kalahari Khoe, Kxoe branch 10
Naro Khoe-Kwadi West Kalahari Khoe, Naro branch 10
G|ui Khoe-Kwadi West Kalahari Khoe, G||ana branch 5
G||ana Khoe-Kwadi West Kalahari Khoe, G||ana branch 5
Hai||om Khoe-Kwadi KhoeKhoe 10
Nama Khoe-Kwadi KhoeKhoe 16
Damara Khoe-Kwadi KhoeKhoe 15
Kgalagadi Niger-Congo Bantu 5
Wambo Niger-Congo Bantu 5
Mbukushu Niger-Congo Bantu 4
Tswana Niger-Congo Bantu 5
Himba Niger-Congo Bantu 5
Hadza isolate Hadza 7
Supplementary Table S1: Summary of samples genotyped in this study.
25

### Table 1 (Page 42)

| Tuu
Tuu
Tuu
Kx’a
Kx’a
Kx’a
Kx’a
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Khoe-Kwadi
Niger-Congo
Niger-Congo
Niger-Congo
Niger-Congo
Niger-Congo
isolate | Taa-Lower Nossob
Taa-Lower Nossob
Taa-Lower Nossob
Northwest Ju
Southeast Ju
Southeast Ju
}Hoan
East Kalahari Khoe
East Kalahari Khoe
West Kalahari Khoe, Kxoe branch
West Kalahari Khoe, Naro branch
West Kalahari Khoe, G||ana branch
West Kalahari Khoe, G||ana branch
KhoeKhoe
KhoeKhoe
KhoeKhoe
Bantu
Bantu
Bantu
Bantu
Bantu
Hadza | 6
6
8
13
16
9
7
10
10
10
10
5
5
10
16
15
5
5
4
5
5
7 |
| --- | --- | --- |

## Page 43

Sample Population
BOT6.090 Ju|’hoan South
NAM066 Ju|’hoan South
NAM051 Ju|’hoan South
BOT6.025 Taa North
BOT6.255 Shua
NAM189 !Xuun
NAM195 !Xuun
BOT6.004 Kgalagadi
DR000071 Hadza
BOT6.058 Naro
Supplementary Table S2: Individuals removed from analysis.
26

## Page 44

Target Population “Mixing” populations Minimum f Z-score
3
Khwe Ju|’hoan North, Yoruba -0.005 -38.7
Hai||om Ju|’hoan North, Yoruba -0.005 -33.9
Tshwa Ju|’hoan North, Yoruba -0.005 -29.9
Shua Ju|’hoan North, Yoruba -0.004 -28.8
Tswana Yoruba, Taa West -0.004 -23.8
!Xuun Ju|’hoan North, Yoruba -0.004 -20.3
G||ana Ju|’hoan North, Yoruba -0.005 -21.3
Kgalagadi Ju|’hoan North, Yoruba -0.002 -8.4
Naro Ju|’hoan North, Taa North -0.0006 -4.4
Mbukushu Ju|’hoan North, Yoruba -0.0008 -3.9
Taa West Taa North, Kgalagadi -0.0008 -3.6
Wambo Ju|’hoan North, Yoruba -0.0003 -1.6
Supplementary Table S3: Three-population tests for treeness. We performed three-
population tests on all possible combinations of populations. Shown are all populations with at
least one negative f statistic, the names of the putative mixing populations that give rise to the
3
minimum f statistic, the value of the statistic, and the Z-score. A Z-score of less than -3 corre-
3
sponds to a p-value of less than 0.001. The populations labeled as “mixing” populations are those
that give the minimum f statistic, and are not necessarily the populations that actually mixed
3
historically.
27

### Table 1 (Page 44)

| Ju|’hoan North, Yoruba
Ju|’hoan North, Yoruba
Ju|’hoan North, Yoruba
Ju|’hoan North, Yoruba
Yoruba, Taa West
Ju|’hoan North, Yoruba
Ju|’hoan North, Yoruba
Ju|’hoan North, Yoruba
Ju|’hoan North, Taa North
Ju|’hoan North, Yoruba
Taa North, Kgalagadi
Ju|’hoan North, Yoruba | -0.005
-0.005
-0.005
-0.004
-0.004
-0.004
-0.005
-0.002
-0.0006
-0.0008
-0.0008
-0.0003 | -38.7
-33.9
-29.9
-28.8
-23.8
-20.3
-21.3
-8.4
-4.4
-3.9
-3.6
-1.6 |
| --- | --- | --- |

## Page 45

Proportion Date of
Population non-Khoisan mixture
ancestry (gen.)
Ju|’hoan North 0.06 35
Ju|’hoan South 0.07 35
Naro 0.09 37
Taa North 0.11 30
Taa West 0.12 10
Taa East 0.17 11
}Hoan 0.23 11
!Xuun 0.25 16
G||ana 0.38 13
Hai||om 0.43 28
Tshwa 0.48 19
Kgalagadi 0.61 23
Shua 0.62 22
Khwe 0.63 25
Tswana 0.76 25
Mbukushu 0.90 14
Damara 0.90 25
Himba 0.93 41
Wambo 0.93 NA
Supplementary Table S4: Admixture parameters in southern Africa. We report the
admixture proportions and times for each southern African population displayed in Figure 2 in the
main text.
28

### Table 1 (Page 45)

| 0.06
0.07
0.09
0.11
0.12
0.17
0.23
0.25
0.38
0.43
0.48
0.61
0.62
0.63
0.76
0.90
0.90
0.93
0.93 | 35
35
37
30
10
11
11
16
13
28
19
23
22
25
25
14
25
41
NA |
| --- | --- |

## Page 46

3 Supplementary Methods
3.1 Data
3.1.1 Sampling
The southern African samples included in this study were collected in various locations in Botswana and
Namibiaaspartofamultidisciplinaryproject,afterethicalclearancebytheReviewBoardoftheUniversity
of Leipzig and with prior permission of the Ministry of Youth, Sport and Culture of Botswana and the
Ministry of Health and Social Services of Namibia. Informed consent was obtained from all donors by
carefully explaining the aims of the study and answering any arising questions with the help of translators
fluent in English/Afrikaans and the local lingua franca; when necessary, a second translation from the local
linguafrancaintothenativelanguageofapotentialdonorwasprovidedbyindividualswithineachsampling
location. Approximately 2ml of saliva were collected in tubes containing 2ml of stabilizing buffer; DNA was
extracted from the saliva with a modified salting-out method [39].
For the purposes of this study, a minimum of 10 unrelated individuals from each linguistic branch of the
three Khoisan language families as given by Gu¨ldemann [40] were selected from the total number of samples
collected in the field, as shown in Supplementary Table S1 and Supplementary Figure S1; only the }Hoan
branch is represented by fewer than 10 individuals due to the small number of samples available. It should
be noted that the “linguistic subgroup” given in the table does not represent the same level of linguistic
relationship for all the populations. The group here called Ju|’hoan North largely corresponds to what is
known as Ju|’hoan, often simply referred to as San in the literature. The Ju|’hoan South are also known as
}Kx’au||’en. Since the dialectal boundaries are as yet uncertain and since both groups partly self-identified
as Ju|’hoan, we here chose geographically defined labels. The HGDP “San” samples were included with the
Ju|hoan North sample, since they clearly stem from this population [18] and empirically we cannot detect
any genetic differentiation between them using the data from this study. For the Khwe, five samples each
of the ||Xokhoe and ||Anikhoe subgroups were combined, while Damara and Nama individuals were chosen
to represent the greatest diversity of traditional subgroups. Since not enough samples were available from
all the different Taa dialects, the subgroups of Taa investigated here were chosen based on both linguistic
and geographic criteria; they do not correspond to any single linguistic unit. The Taa West group includes
speakers of the West !Xoon and !Ama dialects, Taa North comprises speakers of the East !Xoon dialect,
and Taa East includes speakers of the Tshaasi and }Huan dialects. In addition, five samples each from
different Bantu-speaking groups from southwestern Zambia (Mbukushu), Namibia (Himba and Wambo),
and Botswana (Kgalagadi and Tswana) were included.
The Hadza samples are a subset of those from Henn et al. [17] . Genotypes from other populations were
available from other sources, as described below.
3.1.2 Genotyping
Samples were sent to Affymetrix to be genotyped on the Human Origins array. Full details about this array
are in Patterson et al. [19] , but briefly, SNPs were ascertained by identifying heterozygous SNPs in low-
coverage sequencing of single individuals of known ancestry. The SNPs on the array can thus be split into
panelsofSNPsdiscoveredindifferentindividuals. Inallanalyses,weconsideronlyautosomalSNPs. Except
whereotherwisenoted,werestrictourselvestousingthe150,425SNPsdiscoveredinasingleJu|’hoan North
(HGDP “San”) individual. The exception to this are all ROLLOFF analyses (e.g., Figures 2A and 2C in
the main text), where we used all 565,259 SNPs on the array.
29

## Page 47

The Dinka genotypes were taken from Meyer et al. [21] . Genotypes from other populations were
described in Patterson et al. [19] .
3.1.3 Merging data from Lachance et al. [20]
Full genome sequences for five Sandawe, five Hadza, and five Baka/Bakola individuals were obtained via
Complete Genomics by Lachance et al. [20] and merged with publicly available Complete Genomics data
from a number of other individuals. For our purposes, the most important of these samples are 1) the five
Sandawe samples, 2) the five Hadza samples, for comparison to the Hadza we have genotyped, and 3) the
HapMap “YRI” samples, since we have genotyped some of these exact samples on the Affymetrix Human
Origins array (and thus can get estimates of genotype error rate). We extracted the genotypes at each SNP
on the Human Origins array from the Complete Genomics sequences.
TodetectSNPswheregenotyping(oneitherthearrayorviasequencing)performedpoorly,weusedaset
ofeightYoruba(HapMapYRI)samplesthatwerebothsequencedbyCompleteGenomicsandgenotypedon
the Human Origins array. We removed all SNPs where there were any discordant genotypes between these
8 samples. There were 10,888 such SNPs. We then merged the sequenced Hadza and Sandawe samples into
the southern African dataset, again considering only autosomal SNPs. For all analyses involving the Hadza
and Sandawe, we included these Hadza and Sandawe individuals. We used the sequenced YRI samples only
for quality control.
Quality control. Since the Hadza population is quite small, we first used plink [41] to test whether the
two sets of Hadza samples (those genotyped on the array and those genotyped by sequencing) contained
any relatives. We removed one individual that appeared to be the exact same individual in the two samples
(πˆ =0.98 when using the --genome option in plink)
We then wanted to ensure that there were no systematic differences between samples directly genotyped
on the array and those genotyped by sequencing. To look for systematic effects, we used the clustering
algorithm ADMIXTURE [42]. Two populations, the Hadza and the Yoruba/YRI, include some samples
genotyped on the array and some genotyped via sequencing. For both of these populations, there do not
appear to be any substantial differences between samples typed using the two methods (Section 3.2.2, Sup-
plementary Figure S8)
3.1.4 Filtering “outlier” individuals
As described in the main text, for analyses where we grouped individuals into populations, we removed
genetic outliers. To identify individuals that were genetic outliers with respect to their population, we
performed PCA on the genotype matrix using the SNPs ascertained in a single Ju|’hoan North individual
(see Section 3.2). We examined each population in turn, and removed individuals that appear as outliers in
their population (Supplementary Figure S7). A list of all the individuals removed from subsequent analyses
isinSupplementaryTableS2. WefurthermoreexcludedtheG|uipopulation,forwhomwecouldnotidentify
a clear genetic cluster of individuals, since three samples clustered with the southeast Kalahari groups, and
two with the Nama and G||ana.
3.2 Clustering analyses
We performed clustering analysis of the genotype matrix using both PCA [22] and ADMIXTURE [42]. The
latter is a fast implementation of the admixture model of STRUCTURE [43] appropriate for genome-wide
30

## Page 48

data.
3.2.1 PCA
WefirstperformedPCA[22]usingtheSNPsascertainedintheJu|’hoan Northindividualandincludingsome
non-African populations (Supplementary Figure S2). Nearly all of the Khoisan fall along a cline between
the least admixed Khoisan populations and the rest of the African populations. The one major exception is
theNama,whoarescatteredinthePCAplot,indicatingdifferentialrelatednesstonon-Africanpopulations.
We examine this further in Section 3.2.3.
We then considered only the African populations (excluding the Hadza and Sandawe, as we analyze
them separately in Sections 3.7.1 and 3.7.2). As described in the main text, we performed PCA using SNPs
ascertained in either a Ju|’hoan North (HGDP “San” individual) (Supplementary Figure S4A), a Yoruba
(Supplementary Figure S4B), or a French (Supplementary Figure S4C).
Substructure within Khoisan populations. ThePCA(Figure1Binmaintext,orSupplementary
FigureS4A)indicatesthattheTaa WestandtheHai||omaregeneticallysubstructuredpopulations: Halfof
theTaa WestindividualsfallintothesoutheastKalaharicluster,whiletheotherhalfclusterwithNamaand
G||ana. This genetic substructure correlates with a major linguistic boundary: the individuals falling into
thesoutheastKalahariclusterspeaktheWest!XoondialectofTaa,whilethreeoftheotherindividualsspeak
the !Ama dialect. In the case of the Hai||om, four individuals cluster close to the Tshwa and Khwe, while
theotherfiveclusterwiththe!Xuun. ThisgeneticsubstructureintheHai||omreflectsgeographicvariation:
the five individuals that show affnities with the !Xuun come from close to the Angolan border in northern
Namibia, where the two groups are settled in close proximity, while the remaining Hai||om come from the
Etoshaareafurthertothesouthwest. Conversely,forsomegroupstheknownethnolinguisticsubdivisionsdo
not correspond to genetic distinctions. Thus, the Damara sample included four individuals from Sesfontein,
and the Nama sample included five Topnaar from the Kuiseb Valley; these groups are linguistically distinct
[44], butappeargeneticallyindistinguishablefromotherDamaraandNama, respectively. Similarly, thetwo
Khwe subgroups cannot be distinguished from each other in the analyses.
PCA projection. In Supplementary Figure S4C, there is a shift of some Khoe-speaking populations
on the y-axis. For the Nama, we show in Section 3.2.3 that this is due to recent European ancestry. For
the other populations, we speculate that this may be due to eastern African ancestry. We performed a
PCA projection where we first ran the analysis using only the Ju|’hoan North, Yoruba, and Dinka, and
then projected the remaining Khoisan populations on the identified PCs. In this analysis, the Yoruba
represent western African populations and the Dinka represent eastern African populations. This analysis
was performed on the French-ascertained SNPs, as these are the SNPs where a potential eastern African
signal in some of the southern Africans is seen in Supplementary Figure S4C. The results are shown in
Supplementary Figure S5. The projected samples fall on a line between their Bantu-speaking neighbors and
theJu|’hoan North. Thissuggeststhatthemajorityofthevariationinadmixtureinthesepopulationsisdue
to variable levels of admixture with their neighbors. However, we cannot rule out some level of admixture
with non-Bantu-speaking populations.
3.2.2 ADMIXTURE analyses
WeranADMIXTURE[42]onalloftheindividualsintheAfricanpopulations(includingtheFrench,Basque,
HanandOroqenasreferencenon-Africanpopulations). Topreparethedataforanalysis,wethinnedSNPsin
31

## Page 49

LD using plink [41], as suggested by the authors [42]. The precise command was --indep-pairwise 50 10
0.1. ResultsfordifferentnumbersofclustersareshowninSupplementaryFigureS8(thisisforthecombined
set of southern and eastern African populations). We recapitulate previous results showing that clustering
analyses find correlations in allele frequencies between southern African populations and the Mbuti, Biaka,
Hadza, and Sandawe [16] (see K = 4). We additionally recapitulate (at K = 8) the PCA result showing
detectable structure between northwestern and southeastern Kalahari populations.
3.2.3 European ancestry in the Nama
The positions of the Nama individuals in Supplementary Figure S2 are suggestive of post-colonial European
admixture, in accordance with historic documentation of European ancestry in some Nama groups [23] . To
testthis, weusedfour-populationtests[26]oftheform[[Yoruba, X],[Han, French]], whereXisanysouthern
African population. A positive f statistic indicates gene flow between X and a population related to the
4
French (or alternatively gene flow between populations related to the Yoruba and Han). The most strongly
positive f statistic in the southern African populations is for the Nama (Supplementary Figure S6A), as
4
expected if they have experienced European admixture. To confirm the direction of this gene flow, we used
ROLLOFF [27] to test if there is detectable admixture LD in the Nama. If we use the Ju|’hoan North and
the French as the putative mixing populations, there is clear admixture LD (Supplementary Figure S6B),
we date this mixture to approximately five generations (≈150 years) ago. The single exponential curve does
not perfectly fit the curve in the Nama at shorter genetic distances (Supplementary Figure S6B), indicating
that they were likely admixed with some non-Khoisan group at the time of the European admixture. This
means that the extremely recent inferred date of admixture in the Nama (five generations) may still be a
slight overestimate.
Interestingly, a few of the Khoe-speaking populations have slightly positive f statistics in this com-
4
parison, and in the Shua the f statistic is significantly greater than zero. We speculate that some of the
4
Khoe-speaking populations have a low level of east African ancestry, and that the relevant east African
population was itself admixed with a western Eurasian population. The Shua also show a detectable signal
of admixture LD, though we estimate the admixture date as much older (44 generations). This potential
signal of potential east African ancestry specifically in Khoe-speaking populations is of particular interest in
the light of the hypothesis that the Khoe-Kwadi languages were brought to southern Africa by a pre-Bantu
pastoralist immigration from eastern Africa [40].
GiventhattheNamaaretheonlypastoralistKhoisangroupincludedinourdataset,theirrelationshipto
the other Khoisan populations is of particular interest. Unfortunately, the recent European admixture they
have undergone prevents us from including them in further analyses. However, as shown by the PCA based
onJu|hoanSNPs(Fig. 1Bin maintext), the Namadonotstand outamongtheother Khoisanpopulations,
notwithstanding their distinct life-style. Rather, they cluster closely with Tshwa and G||ana foragers, who
also speak languages belonging to the Khoe-Kwadi family, on a cline leading to the southeastern Kalahari
cluster. TowhatextentthisgeneticproximityoftheNamatoforaginggroupsisduetoextensiveadmixture
between immigrating pastoralists and resident foragers [40] or rather to a cultural diffusion of pastoralism
to indigenous hunter-gatherers [33] cannot be addressed at this point.
3.3 Three- and four-population tests
Three- and four-population tests for admixture are described most thoroughly in Reich et al. [26] and
Patterson et al. [19] . We used the implementation of f and f statistics available as part of the TreeMix
3 4
32

## Page 50

package [36] . In all cases standard errors for f−statistics were calculated in blocks of 500 SNPs (i.e. -K
500).
Asignificantlynegativef statisticisevidenceforadmixtureinthetestedpopulation. Weperformedall
3
possible three-population tests on the southern African dataset after removing outliers; all populations with
negative f statistics are shown in Supplementary Table S3. Note that genetic drift since admixture reduces
3
the power of this test [26] .
In various places, we use four-population tests. In these tests, of the form f (A,B;C,D) (where A, B,
4
C, and D are populations) a significantly positive statistic indicates gene flow between populations related
to either A and C or B and D, and asignificantly negative statistic indicatesgene flow between populations
related to A and D or B and C.
3.4 Using the decay of linkage disequilibrium to test for historical admixture
3.4.1 Motivation
Acommonapproachtolookingforhistoricaladmixtureinapopulationistouseclusteringanalyseslikethose
implemented in STRUCTURE [43] and PCA [22] . These are useful approaches to summarizing the major
componentsofvariationingeneticdata. Moreformally,theseapproachesattempttomodelthegenotypesof
each sampled individual as a linear combination of unobserved allele frequency vectors. These vectors (and
thebestlinearcombinationofthemforapproximatingthegenotypesofeachindividual)aretheninferredby
somealgorithm. PCAandSTRUCTURE-likeapproachesdifferonlyinhowtheapproximationischosen[45].
Inapplicationstopopulationhistory,theinferredallelefrequencyvectorsareofteninterpretedas“ancestral”
frequencies from some set of populations (in the STRUCTURE-like framework), and the linear combination
leading to an individual’s genotype as “admixture” levels from each of these populations. However, the
inferred populations need never have existed in reality. Consider two historical scenarios: 1) an individual
with50%ancestryfromapopulationwithanallelefrequencyof1and50%ancestryfromapopulationwith
anallelefrequencyof0; and2)anindividualwith100% ancestryfromapopulationwithanallelefrequency
of 0.5. From the point of view of a clustering algorithm, these two scenarios are identical.
Theabovehypotheticalsituationprovidessomeintuitionforsituationswhereclusteringapproachesmight
mislead. ConsiderthesituationdepictedinSupplementaryFigureS9A.Here,therearetwopopulationsthat
splitapart3,200generationsinthepast. Then,40generationsinthepast,10%ofoneofthepopulationswas
replacedbytheother(thesimulationcommandisgiveninSection3.4.3). Wenowsample20individualsfrom
eachpopulationinthepresentdayandrunADMIXTURE[42]. Withtheaboveintuition,itisnotsurprising
that the algorithm does not pick up the simulated admixture event (Supplementary Figure S9B,D).
Our goal here is to find a method that does detect admixture in this simple situation, and to estimate
the admixture proportions. To do this, we will use the decay of linkage disequilibrium (LD) rather than the
allele frequencies alone. Some aspects here are motivated by clustering approaches that use LD information
[46, 47], and a related approach is taken by Myers et al. [48].
3.4.2 Methods
Consider a population C, which has ancestry from two populations (A and B) with admixture proportions
α and 1−α. Now consider two loci separated by a genetic distance of x cM, and let the allele frequencies at
these loci in population A be fA and fA, respectively. Define fB, fB, fC, and fC analogously. In a given
1 2 1 2 1 2
population (say A), define the standard measure of linkage disequilibrium DA = fA −fAfA, where fA is
12 12 1 2 12
the frequency of the haplotype carrying both alleles 1 and 2 in population A. Suppose populations A and B
33

## Page 51

are in linkage equilibrium, so that DA = DB = 0. Now let C result from admixture between populations
12 12
A and B, and for the moment assume infinite population sizes. At time t generations after admixture, then
[49]:
DC(t)=α(1−α)e−tx[fA−fB][fA−fB]. (S1)
12 1 1 2 2
Since fC =αfA+(1−α)fB, we can now write fB = f 1 C−αf 1 A . We thus have:
1 1 1 1 1−α
α
DC(t)= e−tx[fA−fC][fA−fC] (S2)
12 1−α 1 1 2 2
Now assume we have genotyped m SNPs in n haplotypes from population A and n haplotypes from
A C
population C. We need to estimate both allele frequencies and linkage disequilibrium in C; we do this by
splitting the population in half. Let Dˆ be the estimated amount of linkage disequilibrium between SNPs
ij
i and j in population C (using one half of the population), fˆC be the estimated allele frequency of SNP i
i
in population C (from the other half of the population), fˆA be the estimated allele frequency of SNP i in
i
population A (in practice a population closely related to A rather than A itself), and δˆ be fˆA −fˆC. We
i i i
now split pairs of SNPs into bins based on the genetic distance between them. We use a bin size of 0.01 cM.
In each bin, we calculate the regression coefficient βˆ from a regression of Dˆ on δˆδˆ . If we let s be the set
x i j
of all pairs {i,j} of SNPs in bin x, then
(cid:80) δˆδˆ Dˆ
i j ij
βˆ = {i,j}∈s . (S3)
x (cid:80) δˆ2δˆ2
i j
{i,j}∈s
This is a downwardly-biased estimate of β due to finite sample sizes, since E[δˆ2] (cid:54)= δ2. To correct for
x i i
this, note that δ2 is simply an f statistic [26] , and δˆ2 is the biased version of the f statistic. Now call fˆ
i 2 i 2 2
the biased estimate of the f distance between A and C and fˆ(cid:63) the unbiased estimate of this distance (from
2 2
Reich et al. [26] ). We can calculate a corrected version of the regression coefficient, which we call βˆ(cid:63):
x
fˆ2
βˆ(cid:63) =βˆ 2 . (S4)
x x fˆ(cid:63)2
2
Now, returning to the population genetic parameters, recall that (from Equation S2):
α
β = e−tx. (S5)
x 1−α
We thus fit an exponential curve to the decay of the regression coefficient with genetic distance using the
nls()functioninR[50]. ToremovetheeffectsofLDintheancestralpopulations,weignoredistancebinsless
than 0.5 cM. The amplitude of this curve is an estimate of α , and the decay rate is an estimate of t. The
1−α
interpretation of the amplitude in terms of the admixture proportion relies heavily on the assumption that
population A has experienced little genetic drift since the admixture event, and so may not be applicable in
all situations (we show below via simulations that this approximation performs well in a situation like that
of the Khoisan).
3.4.3 Simulations
The above theory is valid in the absence of drift and in the presence of phased haplotypes. To test how
well this works in more realistic situations, we performed coalescent simulations using macs [51]. We simu-
34

## Page 52

lated two populations that diverged 3,200 generations ago, each of which has an effective population size of
10,000. One population then mixes into the other 40 generations ago with some admixture fraction α. The
parameters were chosen to be reasonable for the Yoruba and Ju|’hoan North. We simulated α of 0, 0.1, and
0.2. The macs parameters were (for e.g., α=0.1):
macs 80 100000000 -t 0.0004 -r 0.0004 -I 2 40 40 -em 0.001 2 1 4000 -em 0.001025 2 1 0 -ej
0.08 2 1
To mimic the effects of uncertain phasing, we randomly combined the simulated chromosomes into
diploids,andre-phasedthemusingfastPHASE[52]. Wethenusedtheabovemodeltoestimatetheadmixture
proportions. Wesimulatedfivereplicatesofeachα,andaveragedtheresultingcurves(SupplementaryFigure
S10). Weseethatthecurvesareapproximatelythosepredictedbytheory,thoughtheyslightlyoverestimate
the true mixture proportions. At higher mixture proportions (30% or 40%), phasing errors become a major
problemandαisseverelyunderestimated(notshown). Tworepresentativesimulationsofα=0.1areshown
in Supplementary Figures 9C,E for comparison to the results from ADMIXTURE.
3.4.4 Application to the Khoisan
WethenappliedthisproceduretothefiveKhoisanpopulationsthatdonotshowsignificantevidenceforad-
mixturefromthree-populationtests(SupplementaryTableS3). ThesearetheJu|’hoan North,Ju|’hoan South,
}Hoan, Taa North, and Taa East. We phased the merged dataset using fastPHASE, combining all popu-
lations (using 20 states in the HMM; i.e. K = 20). Genetic distances between SNPs were taken from the
HapMap [53] (all genetic maps are highly correlated at the scale we are considering). We used the Yoruba
as a reference non-Khoisan population, and use the admixed population itself as the other reference (as
described in the theory presented above). All LD decay curves for these populations are shown in Supple-
mentary Figure S11. All five Khoisan populations show a clear curve; we estimate that the Ju|’hoan North
are the least admixed population, with approximately 6% non-Khoisan ancestry.
A potential concern is that demographic events other than admixture (like population bottlenecks) may
also lead to substantial LD in some populations. This concern arises because we use the Khoisan pop-
ulation twice in Equation S2 (population C)–both to calculate allele frequencies and to calculate linkage
disequilibrium. Thoughwehaveuseddifferentindividualsforthesetwocalculations, therecouldbeunmod-
eled relationships between the individuals in the two sets. To test the robustness of the curves, we used
ROLLOFF [27] . In ROLLOFF, the target population is used only to calculate LD, and two other popu-
lations are used as representatives of the putative mixing populations; see Moorjani et al. [27] for details.
While demographic effects in the target population may influence LD, under the null model that the target
population is unadmixed, the influence on LD will not be correlated to differences in allele frequencies be-
tween two unrelated populations. Results for using the Ju|’hoan North as the target population and various
other pairs of populations as the mixing populations are shown in Supplementary Figure S12. There is a
clear exponential decay of LD in nearly all cases. For example, the level of LD between two distant SNPs
in the Ju|’hoan North is correlated with the divergence of those SNPs between the Yoruba and the French
(Supplementary Figure S12); this is not expected if the Ju|’hoan North are unadmixed.
3.4.5 The f ratio test in the presence of admixed ancestral populations
4
The f ratio test was introduced in Reich et al. [26] as a method to estimate mixture proportions in an
4
admixed group. In our case, imagine we had samples from Chimpanzee (C), Dinka (D), Yoruba (Y), an
35

## Page 53

unadmixed Khoisan population (S), and an admixed Khoisan population (X). In this setup, the chimpanzee
is an outgroup, the Yoruba and population “S” represent populations related to the admixing populations,
and the Dinka are a population that split from the Yoruba before the admixture. Following the derivation
in Reich et al. [26] , if we let α be the amount of Yoruba-like ancestry in population X:
f (C,D;X,Y)
4 =1−α (S6)
f (C,D;S,Y)
4
However,wedonothavesamplesfromS;instead,weonly havesamplesfromadmixedKhoisanpopulations.
Now let α be the fraction of Yoruba-like ancestry in population X, and α be the fraction of Yoruba-like
1 2
ancestry in population S. If we assume the Yoruba-like mixture into X and S occurred from the same
population, then:
f (C,D;X,Y) 1−α
4 = 1 (S7)
f (C,D;S,Y) 1−α
4 2
so
f (C,D;X,Y)
α =1−(1−α ) 4 . (S8)
1 2 f (C,D;S,Y)
4
Ofcourse,usingthisapproachrequiresanindependentmethodforcalculatingα . WeusetheJu|’hoan North
2
aspopulationS,andestimateα fromthelinkagedisequilibriumpatternsasdescribedintheprevioussection.
2
3.5 Estimating mixture dates with ROLLOFF
WeusedROLLOFF[27]toestimateadmixturedatesforallsouthernAfricanpopulations. Todothisweset
the Ju|’hoan North and Yoruba as the two mixing populations (note that the date estimates in ROLLOFF
arerobusttoimproperspecificationofthemixingpopulations[27]),andranROLLOFFoneachpopulation
separately (Supplementary Figure S13). We generated standard errors on the date estimates by performing
a jackknife where we drop each chromosome in turn, as in Moorjani et al. [27] . In this analysis, we used all
the SNPs on the genotyping chip, and genetic distances from the HapMap [53] (all genetic maps are highly
correlated at the scale we are considering). For the Ju|’hoan North, we used half the sample to estimate
allele frequencies and half to estimate LD, as in Section 3.4.
ROLLOFF estimates a single date of admixture, while in reality there may be multiple waves (or con-
tinuous) admixture in the history of a population. In these scenarios, the rate at which admixture LD
decays is no longer an exponential curve, but instead a mixture of multiple exponential curves with different
decay rates. We thus examined the residual fit from fitting a single exponential curve to the LD decay in
each population (Supplementary Figure S14). For a few populations, a single exponential curve does not
completelydescribethedata,especiallyatshortergeneticdistances. Thisimpliesthatforsomepopulations,
most notably the !Xuun, G||ana, Taa East and Taa West, there was likely some admixture before the date
estimated by ROLLOFF.
3.6 Estimating population split times
3.6.1 Motivation
Consider two populations, X and Y. These populations split at time t generations in the past, and our goal
istoestimatetfromgeneticdata(inourcase,SNPs). Therearetwomainapproachesthathavebeenapplied
tothisprobleminthepast. Thefirstapproachisbasedontheobservationthatitisoftenimpossibletowrite
36

## Page 54

downtheprobabilityofseeinggeneticdataunderagivendemographicmodel,butitisquiteeasytosimulate
dataunderessentiallyanydemographicmodel. Itisthuspossibletoidentifydemographicparameterswhich
generate simulated data approximately similar to those observed. This is what is now called approximate
Bayesiancomputation(seePritchardetal. [43]andBeaumontetal. [54]foramoreformaldescription),and
this approach has been applied to estimating split times between populations in a number of applications
(e.g. [55–57]).
The other approach to this problem does not rely on simulations, but uses an explicit expression for the
joint allele frequency spectrum in two populations under a given demographic history [58]. The joint allele
frequencyspectrumisinfluencedbyanumberofdemographicparameters,includingtheeffectivepopulation
sizes of the populations, the time at which they split, and other considerations; Gutenkunst et al. [58]
estimate all of these parameters.
In both approaches, the demographic history of the populations modeled is assumed to be simple–a
constantpopulationsize,exponentialgrowth,orbottleneckmodels(orsomecombinationthereof)arepopular
due to mathematical convenience. However, true population history is almost certainly more complex than
can be modeled. For estimates of population split times, however, the population demography is a nuisance
parameter, and we do not wish to estimate it. We will attempt to estimate split times with an approach
that, in principle, is valid in situations of arbitrary demographic complexity (with some caveats to come
later). The approach we will take is most similar in spirit to Gutenkunst et al. [58], but tailored specifically
to our data. The main idea is roughly as follows: after the split of two populations, a given chromosome
from one of the populations accumulates mutations at a clock-like rate. We wish to count those mutations,
and convert this count to absolute time.
3.6.2 Methods
The demographic setting for the model is presented in Supplementary Figure S15A. We have an outgroup
population O, and two populations whose split time t we wish to estimate, X and Y. The population
ancestral to X and Y is called A. An important modeling assumption is that after populations split, there
is no migration between then. Now imagine we have identified a number of sites that are heterozygous
in a single individual from Y (in applications later on, this will be the Ju|’hoan North; recall that this is
the exact ascertainment scheme used on the Human Origins array). Looking backwards in time, these are
simply all the sites where a mutation has occurred on either of the two chromosomes before they coalesce,
and can be split into two groups–the mutations that arose on the lineage to Y (these are the red stars in
Supplementary Figure S15A) and those that did not (and thus were polymorphic in A; these are the black
starsinSupplementaryFigureS15A,andtheallelicspectruminAisshowninSupplementaryFigureS15B)).
Now consider the allele frequencies in X. The new mutations that arose on the lineage to Y are of course
not polymorphic in X, which leads to a peak of alleles with frequency zero in both the ancestral population
and in X (Supplementary Figure S15B,C). More formally, let f(x) be the allelic spectrum in A conditional
on ascertainment in a single individual from Y. This spectrum can be split into two parts:

λ, if x=0
f(x)= (S9)
(1−λ)g(x), otherwise
where λ is the fraction of SNPs that were non-polymorphic in A (i.e., that arose on the lineage to Y), and
g(x) is the polymorphic frequency spectrum. The key parameter for our purposes is λ. If population sizes
are constant, g(x) is linear, but in more complex situations can take other forms [59]. We assume g(x) is a
quadratic of the form ax2+bx+c. This form is motivated by the fact that the observed allelic spectra are
37

## Page 55

not linear (and thus inconsistent with a constant population size), so we took the next most complicated
model, which seems approximately appropriate in practice (see e.g. Supplementary Figure S19A).
Now we need to write down the (conditional) allelic spectrum in X. To do this, we use the diffusion
approximation to genetic drift. Let τ be the drift length (on the diffusion timescale) between A and X (we
show later on how this can be estimated). Now we can write down the allelic spectrum in X, h(x):
1
(cid:90)
h(x)= f(y)κ(cid:63)(x;y,τ)dy (S10)
0
where κ(cid:63)(x;y,τ) is the probability that an allele at frequency y in A is now at frequency x in X, given
τ. This is closely related to the Kimura transition probability [60], which we call κ(x;y,τ). However, the
Kimura transition probability is the polymorphic transition probability, while we want the probabilities of
fixation as well:
 1 1
 1− (cid:82)
0
κ(z;y,τ)dz)−(y− (cid:82)
0
zκ(z;y,τ)dz), x=0
κ(cid:63)(x;y,τ)= κ(x;y,τ), 0 <x< 1 (S11)

y−
(cid:82)
1
zκ(z;y,τ)dz x=1
0
Now we can write down a likelihood for observed data. Let n be the number of SNPs, let m be the
i
number of sampled alleles in X at SNP i, and let cD be the number of counts of the derived allele at SNP
i
i. Let cD (with no subscript) be the vector of counts of derived alleles.The likelihood for the data is then:
1
n (cid:90)
(cid:89)
l(cD|λ,g(x))= Bin(cD;m,p)h(p)dp (S12)
i
i=10
where Bin(cD;m,p) is the binomial sampling probability. This likelihood can be calculated using numerical
i
integration. We now have three parameters to estimate: two parameters of the polymorphic spectrum in
the ancestral population (a, and b; c is just a scaling factor, which we fix as 1; we normalize the spectrum
so that it is a true probability distribution), and λ. We start with a linear ancestral spectrum and optimize
eachparameterinturnuntilconvergence. Tocalculatestandarderrorsoftheestimatesoftheseparameters,
we perform a block jackknife [26] in blocks of 500 SNPs.
3.6.3 Estimation of τ
Animportantparameterinthismodelisτ,theamountofgeneticdrift(indiffusionunits)thathasoccurred
onthebranchfromAtoX. Allthecomplexityofthechangesinpopulationsizeonthisbranchareabsorbed
into this parameter. Formally:
t
(cid:90) 1
τ = ds (S13)
2N(s)
0
where N(s) is the effective population size at time s in X. To estimate this, we rely on SNPs ascertained by
virtue of being heterozygous in a single individual in an outgroup population (in applications later on this
will be the Yoruba). At a given such SNP, let the derived allele frequency be a in A, x in X, and y in Y.
Now consider the following quantities:
N =x(1−x) (S14)
38

## Page 56

and
D =x(1−y). (S15)
Now consider the expectations of these quantities. For N, this is simply the expected heterozygosity; for a
coalescent derivation see Wakeley [61]:
E[N]=E[x(1−x)] (S16)
=a(1−a)e−τ
and for D, since x and y are conditionally independent given a:
E[D]=a(1−a). (S17)
We now need unbiased estimators of N and D. Let there be n SNPs in the panel used for calculating τ, let
xˆ be the estimated frequency of the derived allele at SNP i in population X, and let yˆ be the estimated
i i
frequency of the derived allele at SNP i in population Y. An estimator of N (which we call Nˆ) is:
n
Nˆ =B + 1 (cid:88) xˆ (1−xˆ ) (S18)
x n i i
i=1
where B is a correction to make this an unbiased estimator (the calculation for B is Equation 4 from the
x x
Supplementary Material in Pickrell and Pritchard [36] ). The estimator of D is the trivial one:
n
Dˆ = 1 (cid:88) xˆ (1−yˆ) (S19)
n i i
i=1
We thus have an estimate of τ:
(cid:18) Nˆ(cid:19)
τˆ=−log . (S20)
Dˆ
3.6.4 Calibration
Oncewe’veestimatedλ, wewouldliketoconvertthistot. λistheproportionofallSNPsascertainedusing
two chromosomes in population Y that arose on the lineage specific to Y (Figure 15A), and can thus be
written in terms of the total number of all mutations specific to these chromosomes (both the red and black
mutations in Figure 15A) and the number of these that arose since t (the red mutations in Figure 15A).
The former is simply the heterozygosity in population Y (call this h), and the latter is 2tµ, where µ is the
mutation rate. This assumes that the two chromosomes do not coalesce before t, which is a fair assumption
in our case where the estimated drift on the Ju|’hoan North lineage is small. We can thus write:
2µt
λ= (S21)
h
and so:
h
t=λ . (S22)
2µ
In practice the ratio of the heterozygosity to the mutation rate must be taken from outside estimates; see
Section 3.6.6 for the specific numbers used in our applications in the Khoisan.
39

## Page 57

3.6.5 Simulations
In this section, we validate the above method using simulations and test its robustness to violations of the
model. In particular, in our case there is gene flow from a non-Khoisan group into the Khoisan, so the
behavior of this method in such situations is quite important. First, using ms [62], we simulated samples
from populations with split times at different depths. All simulations used a demography like that in Figure
15, with samples from an outgroup that split off 3,200 generations in the past, and two populations whose
split time we wish to estimate. The exact ms command used, for a split time of 400 generations in the past,
was:
ms 60 3000 -t 40 -r 40 50000 -I 3 20 20 20 -ej 0.01 3 2 -ej 0.08 2 1
For each simulation, we then generated two sets of SNPs: one ascertained by virtue of being heterozygous
in a single sample from population O (the outgroup), and one ascertained by virtue of being heterozygous
in a single sample from population Y. The procedure for estimating the split time is then as follows:
1. Estimate τ (the drift from A to X) using the SNPs ascertained in O and the method in Section 3.6.3
2. With τ fixed, estimate λ using the SNPs ascertained in Y and the the method from Section 3.6.2
3. Convert the estimated λ to generations using the calibration (the mutation rate has been set for the
simulation and thus is known, and the heterozygosity is estimated in each simulation)
We performed five simulations each at population split times of 400, 1,200, and 2,000 generations. In
all cases, the population split time is well-estimated (Supplementary Figure S16). We then performed sim-
ulations where populations X and Y have experienced some admixture from the “outgroup”. In all cases,
we simulated 5% admixture from O into Y, and variable levels of admixture from O into X. All admixture
occurred 40 generations before present. These numbers were chosen to be appropriate for the Khoisan ap-
plication. The precise ms command (for 5% admixture in X) is:
ms 60 3000 -t 40 -r 40 50000 -I 3 20 20 20 -em 0.002 2 1 2000 -em 0.00205 2 1 0 -em 0.002 3
1 2000 -em 0.00205 3 1 0 -ej 0.01 3 2 -ej 0.08 2 1
We then performed the exact same estimation procedure to get the split times (Supplementary Figure
S17). In all cases, the admixture leads to overestimation of the split time. This is true even when there is
no admixture into X (but only into Y).
3.6.6 Application to the Khoisan
WethenappliedthismethodtodatethesplitofthenorthwesternandsoutheasternKalaharipopulations(the
time of the first split in the southern Africans in Figure 3 in the main text). Some caveats of interpretation
here are warranted. First, all the Khoisan populations have some level of admixture with non-Khoisan
populations. There is thus no single “split time” in their history, and any method (like the one used here)
that estimates a single such time will actually be estimating a composite of several signals. Second, we have
made themodeling assumptionthat historyinvolvespopulations splittingin two withno geneflowafter the
split. More complex demographies are quite plausible, but render the interpretation of a split time nearly
meaningless (if populations continue to exchange migrants after “splitting”, they arguably have not split at
all). We thus consider strong interpretations of split times estimated from genetic data to be impossible,
40

## Page 58

but we nonetheless find the estimates to be useful in constraining the set of historical hypotheses that are
consistent with the data.
For all applications to the Khoisan, the population Y is the Ju|’hoan North, and O (the outgroup) is
the Yoruba. All split times are thus split times between the Ju|’hoan North and another population. We
estimated τ for each population using the set of SNPs ascertained in the Yoruba, and then estimated λ
using the set of SNPs ascertained in the Ju|’hoan North. To convert from λ to t, we need an estimate of
the ratio of the heterozygosity in the Ju|’hoan North to the mutation rate. We took the estimate of this
ratio for the Yoruba from Sun et al. [63] and then used the fact that the heterozygosity in the Yoruba is
95% of that in the Ju|’hoan [21] . Specifically, we averaged this ratio across six Yoruba individuals (from
Supplementary Table 8 in Sun et al. [63]) and multiplied by 1.04 (to account for the estimated factor by
which the heterozygosity in the Ju|’hoan North is greater than that in the Yoruba) to get an estimate of h.
µ
To get from generations to years, we use a generation time of 30 years [64].
The resulting split times are shown in Supplementary Figure S18. We plot these as a function of non-
Khoisan ancestry, as the latter tends to inflate estimates of the split time (Supplementary Figure S17). As
expected,regardlessofthelevelofadmixture,thenorthwesternKalaharigroupshavemorerecentsplittimes
(from the Ju|’hoan North, who are a northwestern Kalahari group) than the southeastern Kalahari groups.
The split time of interest is that with the southeastern Kalahari groups (the red points in Supplementary
Figure S18). The population with the least non-Khoisan ancestry is the Taa North; we show the empirical
and fitted allele frequency spectra for this population in Supplementary Figure S19A, and the simulated
allelefrequencyspectrumfromasimulationwithanolderdateof2,000generationsinSupplementaryFigure
S19B.IntheTaa Northwegetapointestimateof823±99generations(≈25,000±3,000years). However,
the simulations in Supplementary Figure S17 indicate that this is likely an overestimate, and perhaps a
considerable overestimate. We thus conclude that the split between the northwest and southeast Kalahari
groups occurred within the last 30,000 years, and perhaps much more recently than that.
3.7 TreeMix analyses
3.7.1 Analysis of the Hadza
WesoughttounderstandtherelationshipsoftheHadzatothesouthernAfricanpopulations. Todothis, we
selectedpopulationswithlittleadmixturetorepresentthesouthernAfricangroups(theTaa East,Taa North,
Ju|’hoan South, and Ju|’hoan North; see the next section for an analysis of all the Khoisan populations
excluding the Damara), African non-Khoisan groups, and non-African groups. We included the chimpanzee
sequence as an outgroup. We then built a tree of these populations using TreeMix [36] , which fits a tree
to the observed variance-covariance matrix of allele frequencies (Supplementary Figure S20A). The Hadza
do not group with the southern African populations in this analysis; however, they are poorly modeled by
a tree, as seen in the residual fit from the tree (this is the observed covariance matrix subtracted by the
covariance matrix corresponding to the tree model; Supplementary Figure S20B).
WethenallowedTreeMix tobuildthebestgraph, allowingforasingleadmixtureevent(Supplementary
FigureS20C).ThealgorithminfersthattheHadzaareadmixedbetweenapopulationrelatedtothesouthern
African Khoisan groups and a population that is most closely related to the Dinka, a northeastern African
population. ThefractionofKhoisanancestryintheHadzaisestimatedat23±2%(fromablockjackknifein
blocks of 500 SNPs). The residual fit from this graph is shown in Supplementary Figure S20D. The residual
covariance of the Hadza with all populations except the Yoruba are less than three standard errors away
fromthefittedmodel; forthefitofthecovariancebetweentheYorubaandtheHadza,thefitis3.5standard
errors away. Indeed, the Yoruba are particularly poorly fitted in this graph, and the worst fit in this graph
41

## Page 59

is for the fit between the Yoruba and the Chimpanzee (Supplementary Figure S20D). This poor fit for the
Yorubamayindicatearchaicadmixture(indeed, ifweallowTreeMix toestimateasecondmigrationedge, it
estimates admixture from an archaic population into the Yoruba [not shown]). However, other explanations
are possible, and we leave this for future study.
We compared the TreeMix estimate of this Hadza admixture fraction to that obtained by f ancestry
4
estimation. We thus calculated f4(Chimp,Yoruba;Hadza,Dinka) , which is an approximation of the frac-
f4(Chimp,Yoruba;Ju|(cid:48)hoanNorth,Dinka)
tionofJu|’hoan-likeancestryintheHadza(thoughnecessarilyaslightoverestimateduetothenon-Khoisan
ancestryintheJu|’hoan North). Thisestimateis27±1.7%,whichisconsistentwiththeTreeMix estimate.
To ensure that this estimate is reasonable, we replaced the Hadza by the Bantu-speakers from Kenya from
the HGDP (who are an eastern African population not expected to have any Khoisan ancestry) and per-
formed the same analysis. We get an estimate of 3 ± 1.2% Khoisan ancestry, confirming the reliability of
the estimate.
As discussed in the main text, the major caveat to the interpretation of the Hadza result is that a
plausible alternative interpretation for the failure of the tree [Chimp, Ju|’hoan North, [Dinka, Hadza]] is
more archaic gene flow into the ancestors of the Dinka than into the ancestors of the Hadza. There is no
signal of Neandertal or Denisova ancestry in the Dinka [21] , so the source of archaic gene flow would have
to be an undiscovered population. We thus prefer the interpretation that the Hadza share ancestry with the
Khoisan, though we acknowledge the possibility that future work will challenge this interpretation.
3.7.2 Analysis of the Sandawe
TobeginouranalysisoftheSandawe,weperformedthesameanalysesasdonewiththeHadza. Webeganby
building the maximum likelihood tree of populations including the Sandawe using TreeMix (Supplementary
Figure S21A). Like the Hadza, the Sandawe are poorly fitted by a tree (Supplementary Figure S21B), so we
allowed a single migration edge. The inferred migration event is from a population related to the Khoisan,
likewepreviouslysawintheHadza(SupplementaryFigureS21C).TheTreeMix estimateisthattheSandawe
trace about 18% of their ancestry to a population related to the Khoisan.
3.7.3 West Eurasian ancestry in the Sandawe and Hadza.
We noted that the fit of the Sandawe in the TreeMix graph is imperfect (Supplementary Figure S21D). In
particular, the relationship between the Sandawe and the European populations in these data is a poor fit.
On inspection, the Hadza also show a similar signal, but to a lesser extent (Supplementary Figure S20D).
We thus examined the Sandawe and Hadza for evidence of west Eurasian ancestry. We used f statistics
4
of the form [Chimp, X,[French, Han]], where X is either the Sandawe or the Hadza. In both cases, this
tree fails. For the Hadza, this tree fails with a Z-score of -4.2 (p = 1.3×10−5), and for the Sandawe, this
tree fails with a Z-score of -7.2 (p = 3×10−13). Both of these are consistent with west Eurasian (either
European or, more likely, Arabian), gene flow into these populations. To further examine this, we turned to
ROLLOFF. We used Dinka and French as representatives of the mixing populations (since date estimates
are robust to improperly specified reference populations). The results are shown in Supplementary Figure
S22. Both populations show a detectable curve, though the signal is much stronger in the Sandawe than
in the Hadza. The implied dates are 89 generations (≈2500 years) ago for the Hadza and 66 generations
(≈2000years)agofortheSandawe. ThesearequalitativelysimilarsignalstothoseseenbyPaganietal. [65]
in Ethiopian populations. There are two possible historical scenarios that could lead to these signals: either
theHadzaandSandawebothdirectlyadmixedwithawesternEurasianpopulationabout2,000yearsago,or
42

## Page 60

they admixed with an east African population that was itself admixed with a western Eurasian population.
The latter possibility would be consistent with known east African admixture into the Sandawe [16] .
3.7.4 Modification of TreeMix to include known admixture
Since all of the southern African Khoisan populations are admixed with non-Khoisan populations, any
attempt to build a tree relating these populations is complicated by admixture. We wanted to examine
the historical relationships of these populations before the admixture. To do this, we used the composite
likelihood approach of Pickrell and Pritchard [36] , as implemented in the software TreeMix. Briefly, the
approachistobuildagraphofpopulations(whichallowsforbothpopulationsplitsandmixtures)thatbest
fits the sample covariance matrix of allele frequencies [36] . In all analyses, we calculate the standard errors
on the entries in the covariance matrix in blocks of 500 SNPs.
In the original TreeMix algorithm, one first builds the best-fitting tree of populations. However, this
approach is not ideal if there are many admixed populations (as in our application here, where all of the
Khoisan populations are admixed). To get around this, we allow for known admixture events to be incorpo-
rated into this tree-building step. Imagine that there are several populations that we think a priori might
beunadmixed(inourapplications,thesearetheChimpanzee,Yoruba,Dinka,Europeans,andEastAsians).
We first build the best tree of these unadmixed populations using the standard TreeMix algorithm. Now
assume we have an independent estimate of the admixture level of each Khoisan population, and imagine
we know the source population for the mixture.
ToaddaKhoisanpopulationtothetree,foreachexistingbranchinthetree,weputinabranchleading
tothenewpopulation. Wethenforcetheknownadmixtureeventintothegraphwithafixedweight,update
the branch lengths, and store the likelihood of the graph. After testing all possible branches, we keep the
maximum likelihood graph. We then try all possible nearest-neighbor interchanges to the topology of the
graph (as in the original TreeMix algorithm), keeping the change only if it increases the likelihood. We do
this for all populations. Finally, after adding all the populations with fixed admixture weights, we optimize
the admixture weights, and attempt changes to the graph structure where the source populations for the
admixture events are changed.
ToinitializethemigrationweightsforeachKhoisanpopulation,weusedthecorrectedf ratioestimates
4
from Figure 2B in the main text. To initialize the source population for the mixture events, we chose the
Yoruba for all populations except the Hadza and Sandawe, which we initialized as mixing with the Dinka.
We additionally initialized the Hadza and Sandawe as having 5% and 10% ancestry, respectively, related
to the French, for the reasons described in Section 3.7.3 (these proportions were chosen based on rough
examination of the ADMIXTURE plot (Supplementary Figure S8), but are only used for intialization; the
algorithmthenupdatestheseproportionstofitthedata. Thefinalestimatedproportionsare13%and17%,
respectively.
Toobtainameasureofconfidenceintheresultingtree,ifthereareK blocksof500SNPs,weperformeda
bootstrapanalysiswherewerandomlysampleK blocksfromthegenome(withreplacement)andre-estimate
the tree. We ran this bootstrap analysis 100 times, then counted the fraction of replicates supporting each
split in the tree.
3.7.5 Analysis of the Mbuti and Biaka
It has been proposed that the Mbuti and Biaka hunter-gatherers from central Africa were once part of an
Africa-wide hunter-gatherer population [16] . We thus tested whether these populations share the same
signal of relatedness to the Khoisan as we see in the eastern Africans. We started by looking at f statistics
4
43

## Page 61

oftheform[Chimp,Ju|’hoan North,[X,Dinka]],whereXiseithertheMbutiortheBiaka. Recallthatinthe
maintextweshowthat,ifXistheHadzaorSandawe,thistreefailsinawaythatimpliesadmixturebetween
southern and eastern Africa. Here, these trees also fail (for Biaka, Z = 6.4, p = 7.7×10−11; for Mbuti, Z =
3.1, p = 0.001). However, these failures are in the opposite direction to those seen for the eastern Africans.
Instead, these f statistic imply either archaic ancestry in the Mbuti and Biaka, or gene flow between the
4
Khoisan and the Dinka. We examined whether this signal is robust to the choice of ascertainment panel,
instead using the SNPs ascertained in a single Yoruba individual. In this panel, we see the same signals
(for Biaka, Z = 9.6, p < 1×10−12 ; for Mbuti, Z = 6.8, p = 5 ×10−12). The signal of relatedness of
the central Africans to the Khoisan is thus qualitatively and quantitatively different than that seen for the
eastern Africans.
WeusedtheapproachintheprevioussectiontobuildatreerelatingtheMbutiandBiakatotheKhoisan,
like that done in Figure 3 in the main text. As before, we initialized the Khoisan population as having a
fraction of their ancestry related to the Yoruba. We initialized the Mbuti and Biaka as having 37% and
53%,respectively,oftheirancestryrelatedtotheYoruba(thesefractionsareonlyusedforinitialization,but
are then updated in the algorithm). The resulting tree is shown in Supplementary Figure S23. As expected
based on the f statistics, the Mbuti and Biaka do not fall on the same branch as the Khoisan, but instead
4
appear to descend from a population that is an outgroup to everyone else.
44

## Page 62

References
[39] Quinque, D., Kittler, R., Kayser, M., Stoneking, M. & Nasidze, I. Evaluation of saliva as a source of
human DNA for population and association studies. Analytical biochemistry 353, 272–277 (2006).
[40] Gu¨ldemann, T. A linguist’s view: Khoe-Kwadi speakers as the earliest food-producers of southern
Africa. Southern African Humanities 20, 93–132 (2008).
[41] Purcell,S.etal. PLINK:atoolsetforwhole-genomeassociationandpopulation-basedlinkageanalyses.
Am J Hum Genet 81, 559–75 (2007).
[42] Alexander, D. H., Novembre, J. & Lange, K. Fast model-based estimation of ancestry in unrelated
individuals. Genome Res 19, 1655–64 (2009).
[43] Pritchard, J. K., Stephens, M. & Donnelly, P. Inference of population structure using multilocus
genotype data. Genetics 155, 945–59 (2000).
[44] Haacke, W. Linguistic hypotheses on the origin of Namibian Khoekhoe speakers. Southern African
Humanities 20, 163–77 (2008).
[45] Engelhardt, B. E. & Stephens, M. Analysis of population structure: a unifying framework and novel
methods based on sparse factor analysis. PLoS Genet 6 (2010).
[46] Falush,D.,Stephens,M.&Pritchard,J.K. Inferenceofpopulationstructureusingmultilocusgenotype
data: linked loci and correlated allele frequencies. Genetics 164, 1567–87 (2003).
[47] Lawson, D. J., Hellenthal, G., Myers, S. & Falush, D. Inference of population structure using dense
haplotype data. PLoS Genet 8, e1002453 (2012).
[48] Myers, S. et al. LD patterns in dense variation data reveal information about the history of human
populations worldwide. Presented at the 61st Annutal Meeting of The American Society of Human
Genetics (2011).
[49] Chakraborty, R. & Weiss, K. M. Admixture as a tool for finding linked genes and detecting that
difference from allelic association between loci. Proc Natl Acad Sci U S A 85, 9119–23 (1988).
[50] R Development Core Team. R: A Language and Environment for Statistical Computing. R Foundation
forStatisticalComputing, Vienna, Austria(2011). URLhttp://www.R-project.org. ISBN3-900051-
07-0.
[51] Chen, G. K., Marjoram, P. & Wall, J. D. Fast and flexible simulation of DNA sequence data. Genome
Res 19, 136–42 (2009).
[52] Scheet,P.&Stephens,M. Afastandflexiblestatisticalmodelforlarge-scalepopulationgenotypedata:
applicationstoinferringmissinggenotypesandhaplotypicphase. Am J Hum Genet 78, 629–44(2006).
[53] Myers, S., Bottolo, L., Freeman, C., McVean, G. & Donnelly, P. A fine-scale map of recombination
rates and hotspots across the human genome. Science 310, 321–324 (2005).
[54] Beaumont, M. A., Zhang, W. & Balding, D. J. Approximate Bayesian computation in population
genetics. Genetics 162, 2025–35 (2002).
45

## Page 63

[55] Wollstein, A. et al. Demographic history of Oceania inferred from genome-wide data. Curr Biol 20,
1983–92 (2010).
[56] Patin, E. et al. Inferring the demographic history of African farmers and pygmy hunter-gatherers using
a multilocus resequencing data set. PLoS Genet 5, e1000448 (2009).
[57] Lohmueller, K. E., Bustamante, C. D. & Clark, A. G. Methods for human demographic inference
using haplotype patterns from genomewide single-nucleotide polymorphism data. Genetics 182, 217–
31 (2009).
[58] Gutenkunst, R. N., Hernandez, R. D., Williamson, S. H. & Bustamante, C. D. Inferring the joint
demographic history of multiple populations from multidimensional SNP frequency data. PLoS Genet
5, e1000695 (2009).
[59] Keinan, A., Mullikin, J. C., Patterson, N. & Reich, D. Measurement of the human allele frequency
spectrum demonstrates greater genetic drift in East Asians than in Europeans. Nat Genet 39, 1251–5
(2007).
[60] Kimura, M. Solution of a process of random genetic drift with a continuous model. Proceedings of the
National Academy of Sciences of the United States of America 41, 144 (1955).
[61] Wakeley, J. Coalescent Theory: An Introduction (Roberts & Company Publishers, 2009). URL http:
//books.google.com/books?id=x30RAgAACAAJ.
[62] Hudson, R.R. Generating samplesundera Wright-Fisherneutralmodelofgenetic variation. Bioinfor-
matics 18, 337–8 (2002).
[63] Sun, J. X. et al. A direct characterization of human mutation based on microsatellites. Nat Genet
(2012).
[64] Fenner, J. Cross-cultural estimation of the human generation interval for use in genetics-based popula-
tion divergence studies. American journal of physical anthropology 128, 415–423 (2005).
[65] Pagani, L. et al. Ethiopian genetic diversity reveals linguistic stratification and complex influences on
the Ethiopian gene pool. Am J Hum Genet 91, 83–96 (2012).
46

